# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: anaconda 13.0\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2009-10-05 12:04-1000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: anaconda:376
msgid "Press <enter> for a shell"
msgstr ""

#: anaconda:391 rescue.py:324 rescue.py:350 rescue.py:363 rescue.py:439
#: rescue.py:451 text.py:500 loader/cdinstall.c:228 loader/cdinstall.c:232
#: loader/cdinstall.c:248 loader/cdinstall.c:251 loader/cdinstall.c:445
#: loader/cdinstall.c:452 loader/cdinstall.c:517 loader/dirbrowser.c:145
#: loader/driverdisk.c:98 loader/driverdisk.c:243 loader/driverdisk.c:274
#: loader/driverdisk.c:305 loader/driverdisk.c:340 loader/driverdisk.c:359
#: loader/driverdisk.c:371 loader/driverdisk.c:379 loader/driverdisk.c:539
#: loader/driverdisk.c:576 loader/driverselect.c:81 loader/driverselect.c:205
#: loader/hdinstall.c:134 loader/hdinstall.c:279 loader/hdinstall.c:340
#: loader/hdinstall.c:372 loader/hdinstall.c:442 loader/hdinstall.c:485
#: loader/hdinstall.c:498 loader/kbd.c:119 loader/kickstart.c:132
#: loader/kickstart.c:142 loader/kickstart.c:185 loader/kickstart.c:284
#: loader/kickstart.c:316 loader/kickstart.c:497 loader/lang.c:115
#: loader/lang.c:379 loader/loader.c:406 loader/loader.c:442
#: loader/loader.c:485 loader/loader.c:506 loader/loader.c:523
#: loader/loader.c:560 loader/loader.c:1198 loader/loader.c:1382
#: loader/mediacheck.c:46 loader/mediacheck.c:85 loader/mediacheck.c:92
#: loader/mediacheck.c:101 loader/method.c:279 loader/method.c:353
#: loader/modules.c:339 loader/modules.c:351 loader/net.c:582 loader/net.c:953
#: loader/net.c:1587 loader/net.c:1608 loader/net.c:1880 loader/net.c:1902
#: loader/net.c:1914 loader/nfsinstall.c:91 loader/nfsinstall.c:284
#: loader/nfsinstall.c:303 loader/nfsinstall.c:371 loader/telnetd.c:92
#: loader/urlinstall.c:149 loader/urlinstall.c:171 loader/urlinstall.c:242
#: loader/urlinstall.c:400 loader/urlinstall.c:409 loader/urlinstall.c:418
#: loader/urls.c:248 loader/urls.c:328 loader/urls.c:334
#: textw/constants_text.py:44
msgid "OK"
msgstr ""

#: anaconda:398
msgid ""
"You do not have enough RAM to use the graphical installer.  Starting text "
"mode."
msgstr ""

#: anaconda:464
msgid "Starting graphical installation."
msgstr ""

#: anaconda:792
msgid "Would you like to use VNC?"
msgstr ""

#: anaconda:793
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""

#: anaconda:819
msgid "Install class forcing text mode installation"
msgstr ""

#: anaconda:838
msgid "Graphical installation is not available. Starting text mode."
msgstr ""

#: anaconda:846
msgid "DISPLAY variable not set. Starting text mode."
msgstr ""

#: anaconda:1014
msgid "reipl configuration successful => reboot"
msgstr ""

#: anaconda:1017
msgid "reipl configuration failed => halt"
msgstr ""

#: backend.py:148
#, python-format
msgid "Upgrading %s\n"
msgstr ""

#: backend.py:150
#, python-format
msgid "Installing %s\n"
msgstr ""

#: backend.py:170
msgid "Copying File"
msgstr ""

#: backend.py:171
msgid "Transferring install image to hard drive"
msgstr ""

#: backend.py:181
msgid ""
"An error occurred transferring the install image to your hard drive.  This "
"is often cause by damaged or low quality media."
msgstr ""

#: backend.py:185
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""

#: backend.py:189 image.py:288 livecd.py:190 livecd.py:398
#: partIntfHelpers.py:237 text.py:308 text.py:312 yuminstall.py:387
#: yuminstall.py:753 yuminstall.py:867 yuminstall.py:1138 yuminstall.py:1202
#: yuminstall.py:1361 yuminstall.py:1379 iw/autopart_type.py:99
#: iw/autopart_type.py:200 iw/autopart_type.py:320 iw/autopart_type.py:323
#: iw/autopart_type.py:390 iw/autopart_type.py:398 iw/autopart_type.py:427
#: iw/osbootwidget.py:210 iw/osbootwidget.py:219 iw/raid_dialog_gui.py:721
#: iw/raid_dialog_gui.py:760 iw/task_gui.py:63 iw/task_gui.py:180
#: iw/task_gui.py:309 iw/task_gui.py:437 loader/cdinstall.c:228
#: loader/cdinstall.c:517 loader/driverdisk.c:98 loader/driverdisk.c:305
#: loader/driverdisk.c:340 loader/driverdisk.c:371 loader/driverdisk.c:379
#: loader/driverdisk.c:443 loader/hdinstall.c:134 loader/hdinstall.c:340
#: loader/hdinstall.c:442 loader/hdinstall.c:485 loader/hdinstall.c:498
#: loader/kickstart.c:284 loader/lang.c:115 loader/loader.c:406
#: loader/loader.c:523 loader/loader.c:1198 loader/mediacheck.c:46
#: loader/mediacheck.c:85 loader/mediacheck.c:92 loader/method.c:279
#: loader/method.c:353 loader/nfsinstall.c:284 loader/nfsinstall.c:303
#: loader/telnetd.c:92 loader/urlinstall.c:149 loader/urlinstall.c:171
#: loader/urlinstall.c:242 loader/urls.c:328 loader/urls.c:334
#: storage/__init__.py:194 storage/__init__.py:1638 storage/__init__.py:1725
#: textw/netconfig_text.py:282 textw/partition_text.py:186
#: textw/partition_text.py:192 textw/partition_text.py:198
#: textw/partition_text.py:231 textw/partition_text.py:241
#: textw/upgrade_text.py:175 textw/upgrade_text.py:182
msgid "Error"
msgstr ""

#: bootloader.py:53 bootloader.py:209 bootloader.py:216 image.py:82
#: upgrade.py:251 yuminstall.py:1542 yuminstall.py:1577
#: iw/autopart_type.py:210 iw/blpasswidget.py:148 iw/upgrade_swap_gui.py:191
#: iw/upgrade_swap_gui.py:199 iw/upgrade_swap_gui.py:206
#: storage/devicetree.py:144 storage/devicetree.py:172
#: textw/upgrade_text.py:187
msgid "Warning"
msgstr ""

#: bootloader.py:54
msgid ""
"Filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""

#: bootloader.py:58 gui.py:1097 gui.py:1134 gui.py:1262 gui.py:1341
#: image.py:91 kickstart.py:152 livecd.py:197 livecd.py:405 packages.py:119
#: packages.py:127 packages.py:135 packages.py:143 packages.py:155
#: packages.py:164 packages.py:172 packages.py:181 upgrade.py:58
#: upgrade.py:206 yuminstall.py:241 yuminstall.py:667 yuminstall.py:871
#: yuminstall.py:951 yuminstall.py:957 yuminstall.py:1109 yuminstall.py:1134
#: yuminstall.py:1192 yuminstall.py:1356 yuminstall.py:1386
#: storage/__init__.py:95 storage/__init__.py:198 storage/__init__.py:1816
msgid "_Exit installer"
msgstr ""

#: bootloader.py:58 image.py:92 image.py:251 kickstart.py:1358
#: kickstart.py:1397 upgrade.py:58 yuminstall.py:1197 iw/partition_gui.py:1563
#: storage/__init__.py:1817
msgid "_Continue"
msgstr ""

#: bootloader.py:114 platform.py:93 platform.py:320 platform.py:388
#: platform.py:488
msgid "RAID Device"
msgstr ""

#: bootloader.py:143
msgid "Bootloader"
msgstr ""

#: bootloader.py:143
msgid "Installing bootloader."
msgstr ""

#: bootloader.py:210
msgid ""
"There was an error installing the bootloader.  The system may not be "
"bootable."
msgstr ""

#: bootloader.py:217
msgid ""
"No kernel packages were installed on the system.  Bootloader configuration "
"will not be changed."
msgstr ""

#: cmdline.py:55
msgid "Completed"
msgstr ""

#: cmdline.py:63
msgid "In progress"
msgstr ""

#: cmdline.py:86 gui.py:1129 kickstart.py:1238 text.py:345
#, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""

#: cmdline.py:98
msgid ""
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""

#: cmdline.py:114 cmdline.py:121 cmdline.py:128
msgid "Can't have a question in command line mode!"
msgstr ""

#: constants.py:74
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""

#: constants.py:80
msgid " with the provider of this software."
msgstr ""

#: constants.py:84
#, python-format
msgid " against anaconda at %s"
msgstr ""

#: gui.py:106
msgid "An error occurred saving screenshots to disk."
msgstr ""

#: gui.py:117
msgid "Screenshots Copied"
msgstr ""

#: gui.py:118
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""

#: gui.py:161
msgid "Saving Screenshot"
msgstr ""

#: gui.py:162
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr ""

#: gui.py:165
msgid "Error Saving Screenshot"
msgstr ""

#: gui.py:166
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""

#: gui.py:560
msgid "Installation Key"
msgstr ""

#: gui.py:631 text.py:148
msgid ""
"Choose a passphrase for the encrypted devices. You will be prompted for this "
"passphrase during system boot."
msgstr ""

#: gui.py:650 gui.py:658 text.py:186 text.py:196
msgid "Error with passphrase"
msgstr ""

#: gui.py:651 text.py:187
msgid "The passphrases you entered were different.  Please try again."
msgstr ""

#: gui.py:659
msgid "The passphrase must be at least eight characters long."
msgstr ""

#: gui.py:693 text.py:224
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: gui.py:784 gui.py:1341 partIntfHelpers.py:157 partIntfHelpers.py:341
#: text.py:99 text.py:100 iw/account_gui.py:130 loader/dirbrowser.c:145
#: loader/driverdisk.c:244 loader/kickstart.c:316 loader/loader.c:442
#: loader/loader.c:560 textw/constants_text.py:48
msgid "Cancel"
msgstr ""

#: gui.py:796 gui.py:797 gui.py:913 gui.py:914 ui/anaconda.glade.h:2
msgid "_Debug"
msgstr ""

#: gui.py:1131 text.py:347
msgid "Error Parsing Kickstart Config"
msgstr ""

#: gui.py:1171
msgid "default:LTR"
msgstr ""

#: gui.py:1251 text.py:465
msgid "Error!"
msgstr ""

#: gui.py:1252 text.py:466
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""

#: gui.py:1257 image.py:164 image.py:194 packages.py:362 packages.py:367
#: storage/__init__.py:1637
msgid "_Exit"
msgstr ""

#: gui.py:1258 image.py:164 image.py:194 livecd.py:197 yuminstall.py:745
#: yuminstall.py:1134 yuminstall.py:1192 yuminstall.py:1356
msgid "_Retry"
msgstr ""

#: gui.py:1261
msgid "The installer will now exit."
msgstr ""

#: gui.py:1264 storage/partitioning.py:268
msgid "The system will now reboot."
msgstr ""

#: gui.py:1265 image.py:251 packages.py:370 yuminstall.py:1402
msgid "_Reboot"
msgstr ""

#: gui.py:1267
msgid "Exiting"
msgstr ""

#: gui.py:1338 livecd.py:115 text.py:266 upgrade.py:197
msgid "Exit installer"
msgstr ""

#: gui.py:1339
msgid "Are you sure you wish to exit the installer?"
msgstr ""

#: gui.py:1348
#, python-format
msgid "%s Installer"
msgstr ""

#: gui.py:1354
msgid "Unable to load title bar"
msgstr ""

#: gui.py:1416
msgid "Install Window"
msgstr ""

#: image.py:83
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""

#: image.py:156
msgid "Couldn't Mount ISO Source"
msgstr ""

#: image.py:157
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""

#: image.py:185
msgid "Missing ISO 9660 Image"
msgstr ""

#: image.py:186
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""

#: image.py:241
msgid "Required Install Media"
msgstr ""

#: image.py:242
#, python-format
msgid ""
"The software you have selected to install will require the following %"
"(productName)s %(productVersion)s discs:\n"
"\n"
"%(reqcdstr)s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""

#: image.py:251 livecd.py:404 packages.py:367 packages.py:370 upgrade.py:205
#: yuminstall.py:871 yuminstall.py:957 yuminstall.py:1386 yuminstall.py:1402
msgid "_Back"
msgstr ""

#: image.py:289
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: installclass.py:73
msgid "Install on System"
msgstr ""

#: iutil.py:831
#, python-format
msgid ""
"Error: On open, cannot set reIPL method to %(reipl_type)s (%(filename)s: %(e)"
"s)"
msgstr ""

#: iutil.py:842
#, python-format
msgid ""
"Error: On write, cannot set reIPL method to %(reipl_type)s (%(filename)s: %"
"(e)s)"
msgstr ""

#: iutil.py:851
#, python-format
msgid ""
"Error: On close, cannot set reIPL method to %(reipl_type)s (%(filename)s: %"
"(e)s)"
msgstr ""

#: iutil.py:870
#, python-format
msgid "Error: Could not set %(device)s as reIPL device (%(e)s)"
msgstr ""

#: iutil.py:880
#, python-format
msgid "Error: Could not reset loadparm (%s)"
msgstr ""

#: iutil.py:889
#, python-format
msgid "Warning: Could not reset parm (%s)"
msgstr ""

#: iutil.py:899
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from DASD device %s to continue "
"installation"
msgstr ""

#: iutil.py:921
#, python-format
msgid "Error: reading FCP property %(syspath_property)s for reIPL (%(e)s)"
msgstr ""

#: iutil.py:936
#, python-format
msgid "Error: writing FCP property %(reipl_property)s for reIPL (%(e)s)"
msgstr ""

#: iutil.py:951
#, python-format
msgid ""
"Error: writing default FCP property %(reipl_property)s for reIPL (%(e)s)"
msgstr ""

#: iutil.py:963
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from FCP %(device)s with WWPN %"
"(wwpn)s and LUN %(lun)s to continue installation"
msgstr ""

#: iutil.py:970
msgid ""
"After shutdown, please perform a manual IPL from the device now containing /"
"boot to continue installation"
msgstr ""

#: iutil.py:978
msgid "Error determining mount point type"
msgstr ""

#: iutil.py:982
msgid "The mount point /boot or / is on a disk that we are not familiar with"
msgstr ""

#: kickstart.py:97
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  You may "
"examine the output in %(msgs)s.  This is a fatal error and installation will "
"be aborted.  Press the OK button to exit the installer."
msgstr ""

#: kickstart.py:113 kickstart.py:115
msgid "Scriptlet Failure"
msgstr ""

#: kickstart.py:146 rescue.py:240 yuminstall.py:662 yuminstall.py:1104
msgid "No Network Available"
msgstr ""

#: kickstart.py:147
msgid ""
"Encryption key escrow requires networking, but there was an error enabling "
"the network on your system."
msgstr ""

#: kickstart.py:1224
#, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr ""

#: kickstart.py:1227
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr ""

#: kickstart.py:1299 livecd.py:221
msgid "Post-Installation"
msgstr ""

#: kickstart.py:1300
msgid "Running post-installation scripts"
msgstr ""

#: kickstart.py:1316
msgid "Pre-Installation"
msgstr ""

#: kickstart.py:1317
msgid "Running pre-installation scripts"
msgstr ""

#: kickstart.py:1349
msgid "Missing Package"
msgstr ""

#: kickstart.py:1350
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort this installation?"
msgstr ""

#: kickstart.py:1356 kickstart.py:1395
msgid "_Abort"
msgstr ""

#: kickstart.py:1357 kickstart.py:1396
msgid "_Ignore All"
msgstr ""

#: kickstart.py:1387
msgid "Missing Group"
msgstr ""

#: kickstart.py:1388
#, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort this installation?"
msgstr ""

#: kickstart.py:1493
#, python-format
msgid ""
"The kickstart configuration file is missing required information that "
"anaconda cannot prompt for.  Please add the following sections and try "
"again:\n"
"%s"
msgstr ""

#: livecd.py:110
msgid "Unable to find image"
msgstr ""

#: livecd.py:111
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""

#: livecd.py:172
msgid "Copying live image to hard drive."
msgstr ""

#: livecd.py:191
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""

#: livecd.py:222
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes."
msgstr ""

#: livecd.py:399
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""

#: network.py:56
msgid "Hostname must be 255 or fewer characters in length."
msgstr ""

#: network.py:62
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or '0-"
"9'"
msgstr ""

#: network.py:67
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr ""

#: network.py:176
msgid "IP address is missing."
msgstr ""

#: network.py:180
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""

#: network.py:183
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr ""

#: network.py:185
#, python-format
msgid "'%s' is an invalid IP address."
msgstr ""

#: packages.py:114
msgid "Device Resize Failed"
msgstr ""

#: packages.py:115
#, python-format
msgid "An error was encountered while resizing device %s."
msgstr ""

#: packages.py:122
msgid "Device Creation Failed"
msgstr ""

#: packages.py:123
#, python-format
msgid "An error was encountered while creating device %s."
msgstr ""

#: packages.py:130
msgid "Device Removal Failed"
msgstr ""

#: packages.py:131
#, python-format
msgid "An error was encountered while removing device %s."
msgstr ""

#: packages.py:138
msgid "Device Setup Failed"
msgstr ""

#: packages.py:139
#, python-format
msgid "An error was encountered while setting up device %s."
msgstr ""

#: packages.py:150
msgid "Resizing Failed"
msgstr ""

#: packages.py:151
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr ""

#: packages.py:158
msgid "Migration Failed"
msgstr ""

#: packages.py:159
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr ""

#: packages.py:167
msgid "Formatting Failed"
msgstr ""

#: packages.py:168
#, python-format
msgid "An error was encountered while formatting device %s."
msgstr ""

#: packages.py:176
msgid "Storage Activation Failed"
msgstr ""

#: packages.py:177
msgid "An error was encountered while activating your storage configuration."
msgstr ""

#: packages.py:181
msgid "_File Bug"
msgstr ""

#: packages.py:346 packages.py:371
msgid "Warning! This is pre-release software!"
msgstr ""

#: packages.py:347
#, python-format
msgid ""
"Thank you for downloading this pre-release of %(productName)s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %(bugzillaUrl)s\n"
"\n"
"and file a report against '%(fileagainst)s'.\n"
msgstr ""

#: packages.py:362
msgid "_Install anyway"
msgstr ""

#: packages.py:366
msgid "The installer will now exit..."
msgstr ""

#: packages.py:369
msgid "Your system will now be rebooted..."
msgstr ""

#: partIntfHelpers.py:41
msgid "Please enter a volume group name."
msgstr ""

#: partIntfHelpers.py:45
msgid "Volume Group Names must be less than 128 characters"
msgstr ""

#: partIntfHelpers.py:48
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr ""

#: partIntfHelpers.py:53
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: partIntfHelpers.py:63
msgid "Please enter a logical volume name."
msgstr ""

#: partIntfHelpers.py:67
msgid "Logical Volume Names must be less than 128 characters"
msgstr ""

#: partIntfHelpers.py:71
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr ""

#: partIntfHelpers.py:77
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: partIntfHelpers.py:101
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""

#: partIntfHelpers.py:108
msgid "Please specify a mount point for this partition."
msgstr ""

#: partIntfHelpers.py:121 partIntfHelpers.py:128
msgid "Unable To Delete"
msgstr ""

#: partIntfHelpers.py:122
msgid "You must first select a partition to delete."
msgstr ""

#: partIntfHelpers.py:153 partIntfHelpers.py:340 iw/lvm_dialog_gui.py:840
msgid "Confirm Delete"
msgstr ""

#: partIntfHelpers.py:154
#, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr ""

#: partIntfHelpers.py:157 partIntfHelpers.py:341 iw/lvm_dialog_gui.py:843
#: iw/lvm_dialog_gui.py:1376 iw/osbootwidget.py:103 iw/partition_gui.py:1764
#: iw/partition_gui.py:1776
msgid "_Delete"
msgstr ""

#: partIntfHelpers.py:204
msgid "Notice"
msgstr ""

#: partIntfHelpers.py:205
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""

#: partIntfHelpers.py:220
msgid "Format as Swap?"
msgstr ""

#: partIntfHelpers.py:221
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""

#: partIntfHelpers.py:236
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr ""

#: partIntfHelpers.py:241
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""

#: partIntfHelpers.py:249
msgid "Format?"
msgstr ""

#: partIntfHelpers.py:249 iw/partition_gui.py:1561
msgid "_Modify Partition"
msgstr ""

#: partIntfHelpers.py:249
msgid "Do _Not Format"
msgstr ""

#: partIntfHelpers.py:257
msgid "Error with Partitioning"
msgstr ""

#: partIntfHelpers.py:258
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of %"
"(productName)s.\n"
"\n"
"%(errorstr)s"
msgstr ""

#: partIntfHelpers.py:274
msgid "Partitioning Warning"
msgstr ""

#: partIntfHelpers.py:275
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: partIntfHelpers.py:289 iw/partition_gui.py:842
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""

#: partIntfHelpers.py:292
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""

#: partIntfHelpers.py:298
msgid "Format Warning"
msgstr ""

#: partIntfHelpers.py:324
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""

#: partIntfHelpers.py:328
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr ""

#: partIntfHelpers.py:331
msgid "You are about to delete a RAID device."
msgstr ""

#: partIntfHelpers.py:333
#, python-format
msgid "You are about to delete the %s partition."
msgstr ""

#: partIntfHelpers.py:337
#, python-format
msgid "You are about to delete the %(type)s %(name)s"
msgstr ""

#: partIntfHelpers.py:348
msgid "Confirm Reset"
msgstr ""

#: partIntfHelpers.py:349
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr ""

#: platform.py:94 platform.py:97 platform.py:321 platform.py:389
#: platform.py:489 platform.py:492
msgid "Master Boot Record (MBR)"
msgstr ""

#: platform.py:96 platform.py:491
msgid "First sector of boot partition"
msgstr ""

#: platform.py:112
msgid "You have not created a bootable partition."
msgstr ""

#: platform.py:115
msgid "Bootable partitions can only be on RAID1 devices."
msgstr ""

#: platform.py:119
msgid "Bootable partitions cannot be on a logical volume."
msgstr ""

#: platform.py:123
msgid "Bootable partitions cannot be on a RAID device."
msgstr ""

#: platform.py:127 platform.py:131
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr ""

#: platform.py:135 platform.py:140
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr ""

#: platform.py:214
msgid "EFI System Partition"
msgstr ""

#: platform.py:219
msgid "You have not created a /boot/efi partition."
msgstr ""

#: platform.py:225
msgid "/boot is not on an ext2 filesystem."
msgstr ""

#: platform.py:228
msgid "/boot/efi is not EFI."
msgstr ""

#: platform.py:265
#, python-format
msgid "%s must have a bsd disk label."
msgstr ""

#: platform.py:277
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr ""

#: platform.py:323
msgid "PPC PReP Boot"
msgstr ""

#: platform.py:339
msgid "The boot partition must be within the first 4MB of the disk."
msgstr ""

#: platform.py:391 platform.py:394
msgid "Apple Bootstrap"
msgstr ""

#: platform.py:408
#, python-format
msgid "%s must have a mac disk label."
msgstr ""

#: rescue.py:199
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""

#: rescue.py:213
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""

#: rescue.py:232
msgid "Setup Networking"
msgstr ""

#: rescue.py:233
msgid "Do you want to start the network interfaces on this system?"
msgstr ""

#: rescue.py:234 loader/driverdisk.c:483 loader/driverdisk.c:493
#: loader/hdinstall.c:222 textw/constants_text.py:56
msgid "Yes"
msgstr ""

#: rescue.py:234 rescue.py:236 loader/driverdisk.c:483 loader/driverdisk.c:493
#: textw/constants_text.py:60
msgid "No"
msgstr ""

#: rescue.py:241
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""

#: rescue.py:276 rescue.py:345 rescue.py:356 rescue.py:434
msgid "Rescue"
msgstr ""

#: rescue.py:277
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: rescue.py:287 iw/partition_gui.py:746 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/driverdisk.c:444 storage/__init__.py:140
#: storage/devicetree.py:84
msgid "Continue"
msgstr ""

#: rescue.py:287 rescue.py:292
msgid "Read-Only"
msgstr ""

#: rescue.py:287 rescue.py:289 loader/cdinstall.c:248 loader/cdinstall.c:251
#: loader/method.c:326 storage/__init__.py:1637 textw/upgrade_text.py:133
msgid "Skip"
msgstr ""

#: rescue.py:321
msgid "System to Rescue"
msgstr ""

#: rescue.py:322
msgid "Which device holds the root partition of your installation?"
msgstr ""

#: rescue.py:324 rescue.py:328 text.py:470 text.py:472
msgid "Exit"
msgstr ""

#: rescue.py:346
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: rescue.py:357
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: rescue.py:435
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""

#: rescue.py:444
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr ""

#: rescue.py:447
msgid "Rescue Mode"
msgstr ""

#: rescue.py:448
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""

#: rescue.py:461
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr ""

#: text.py:153
msgid "Passphrase for encrypted device"
msgstr ""

#: text.py:166
msgid "Also add this passphrase to all existing encrypted devices"
msgstr ""

#: text.py:197
#, python-format
msgid "The passphrase must be at least %d character long."
msgstr ""

#: text.py:231 ui/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr ""

#: text.py:239 ui/lukspassphrase.glade.h:7
msgid "This is a global passphrase"
msgstr ""

#: text.py:309 text.py:313
msgid "Repository editing is not available in text mode."
msgstr ""

#: text.py:375
#, python-format
msgid "Welcome to %(productName)s for %(productArch)s"
msgstr ""

#: text.py:377
#, python-format
msgid "Welcome to %s"
msgstr ""

#: text.py:379
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr ""

#: text.py:470 loader/net.c:111 loader/net.c:472 loader/net.c:528
#: loader/net.c:671 loader/net.c:679 loader/net.c:1102 loader/net.c:1109
msgid "Retry"
msgstr ""

#: text.py:496
msgid "Cancelled"
msgstr ""

#: text.py:497
msgid "I can't go to the previous step from here. You will have to try again."
msgstr ""

#: upgrade.py:51
msgid "Proceed with upgrade?"
msgstr ""

#: upgrade.py:52
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""

#: upgrade.py:56
msgid "Would you like to continue with the upgrade?"
msgstr ""

#: upgrade.py:184
msgid "Mount failed"
msgstr ""

#: upgrade.py:185
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""

#: upgrade.py:193 upgrade.py:200
msgid "Upgrade root not found"
msgstr ""

#: upgrade.py:194
msgid "The root for the previously installed system was not found."
msgstr ""

#: upgrade.py:201
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""

#: upgrade.py:224
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""

#: upgrade.py:230
msgid "Absolute Symlinks"
msgstr ""

#: upgrade.py:241
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""

#: upgrade.py:247
msgid "Invalid Directories"
msgstr ""

#: upgrade.py:252
#, python-format
msgid "%s not found"
msgstr ""

#: vnc.py:137
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""

#: vnc.py:143
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr ""

#: vnc.py:172
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr ""

#: vnc.py:186
msgid "Connected!"
msgstr ""

#: vnc.py:189
msgid "Will try to connect again in 15 seconds..."
msgstr ""

#: vnc.py:195
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgstr ""

#: vnc.py:206
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""

#: vnc.py:208
msgid "Please manually connect your vnc client to begin the install."
msgstr ""

#: vnc.py:211
msgid "Starting VNC..."
msgstr ""

#: vnc.py:236
msgid "The VNC server is now running."
msgstr ""

#: vnc.py:249
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: vnc.py:254
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: vnc.py:258
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""

#: vnc.py:260
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""

#: vnc.py:282 vnc.py:375
msgid "VNC Configuration"
msgstr ""

#: vnc.py:285 vnc.py:379
msgid "No password"
msgstr ""

#: vnc.py:287 vnc.py:382
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""

#: vnc.py:295 vnc.py:390 textw/userauth_text.py:47
msgid "Password:"
msgstr ""

#: vnc.py:296 vnc.py:391 textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr ""

#: vnc.py:314 vnc.py:413 textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr ""

#: vnc.py:315 vnc.py:414 textw/userauth_text.py:71
msgid "The passwords you entered were different. Please try again."
msgstr ""

#: vnc.py:320 vnc.py:419 textw/userauth_text.py:66
msgid "Password Length"
msgstr ""

#: vnc.py:321 vnc.py:420
msgid "The password must be at least six characters long."
msgstr ""

#: vnc.py:343
msgid "Unable to Start X"
msgstr ""

#: vnc.py:345
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""

#: vnc.py:364
msgid "Start VNC"
msgstr ""

#: vnc.py:365 vnc.py:367
msgid "Use text mode"
msgstr ""

#: yuminstall.py:83
#, python-format
msgid "%s MB"
msgstr ""

#: yuminstall.py:86
#, python-format
msgid "%s KB"
msgstr ""

#: yuminstall.py:88
#, python-format
msgid "%s Byte"
msgstr ""

#: yuminstall.py:132
msgid "Preparing to install"
msgstr ""

#: yuminstall.py:133
msgid "Preparing transaction from installation source"
msgstr ""

#: yuminstall.py:161
#, python-format
msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
msgstr ""

#: yuminstall.py:210
#, python-format
msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgstr ""

#: yuminstall.py:225
msgid "Finishing upgrade"
msgstr ""

#: yuminstall.py:226
msgid "Finishing upgrade process.  This may take a little while."
msgstr ""

#: yuminstall.py:235
msgid "Error Installing Package"
msgstr ""

#: yuminstall.py:236
#, python-format
msgid ""
"A fatal error occurred when installing the %s package.  This could indicate "
"errors when reading the installation media.  Installation cannot continue."
msgstr ""

#: yuminstall.py:308 iw/task_gui.py:82 iw/task_gui.py:322
msgid "Error Setting Up Repository"
msgstr ""

#: yuminstall.py:309
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Please provide the correct information for installing %(productName)s."
msgstr ""

#: yuminstall.py:369
msgid "Change Disc"
msgstr ""

#: yuminstall.py:370
#, python-format
msgid "Please insert %(productName)s disc %(discnum)d to continue."
msgstr ""

#: yuminstall.py:380
msgid "Wrong Disc"
msgstr ""

#: yuminstall.py:381
#, python-format
msgid "That's not the correct %s disc."
msgstr ""

#: yuminstall.py:388
msgid "Unable to access the disc."
msgstr ""

#: yuminstall.py:547
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr ""

#: yuminstall.py:663 yuminstall.py:1105
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""

#: yuminstall.py:743 yuminstall.py:745
msgid "Re_boot"
msgstr ""

#: yuminstall.py:743
msgid "_Eject"
msgstr ""

#: yuminstall.py:754
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: yuminstall.py:802
msgid "Retrying"
msgstr ""

#: yuminstall.py:802
msgid "Retrying download."
msgstr ""

#: yuminstall.py:868
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr ""

#: yuminstall.py:903 yuminstall.py:904
msgid "file conflicts"
msgstr ""

#: yuminstall.py:905
msgid "older package(s)"
msgstr ""

#: yuminstall.py:906
msgid "insufficient disk space"
msgstr ""

#: yuminstall.py:907
msgid "insufficient disk inodes"
msgstr ""

#: yuminstall.py:908
msgid "package conflicts"
msgstr ""

#: yuminstall.py:909
msgid "package already installed"
msgstr ""

#: yuminstall.py:910
msgid "required package"
msgstr ""

#: yuminstall.py:911
msgid "package for incorrect arch"
msgstr ""

#: yuminstall.py:912
msgid "package for incorrect os"
msgstr ""

#: yuminstall.py:926
msgid "You need more space on the following file systems:\n"
msgstr ""

#: yuminstall.py:939
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""

#: yuminstall.py:942
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): %"
"s.\n"
msgstr ""

#: yuminstall.py:949 yuminstall.py:954
msgid "Error Running Transaction"
msgstr ""

#: yuminstall.py:1139
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""

#: yuminstall.py:1175
msgid "Retrieving installation information."
msgstr ""

#: yuminstall.py:1177
#, python-format
msgid "Retrieving installation information for %s."
msgstr ""

#: yuminstall.py:1179
msgid "Installation Progress"
msgstr ""

#: yuminstall.py:1192 textw/constants_text.py:64
msgid "Edit"
msgstr ""

#: yuminstall.py:1203 yuminstall.py:1362
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""

#: yuminstall.py:1380
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""

#: yuminstall.py:1399
msgid "Reboot?"
msgstr ""

#: yuminstall.py:1400
msgid "The system will be rebooted now."
msgstr ""

#: yuminstall.py:1543
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""

#: yuminstall.py:1578
#, python-format
msgid ""
"The arch of the release of %(productName)s you are upgrading to appears to "
"be %(myarch)s which does not match your previously installed arch of %(arch)"
"s.  This is likely to not succeed.  Are you sure you wish to continue the "
"upgrade process?"
msgstr ""

#: yuminstall.py:1631
msgid "Post Upgrade"
msgstr ""

#: yuminstall.py:1632
msgid "Performing post-upgrade configuration"
msgstr ""

#: yuminstall.py:1634
msgid "Post Installation"
msgstr ""

#: yuminstall.py:1635
msgid "Performing post-installation configuration"
msgstr ""

#: yuminstall.py:1864
msgid "Installation Starting"
msgstr ""

#: yuminstall.py:1865
msgid "Starting installation process"
msgstr ""

#: yuminstall.py:1903
msgid "Dependency Check"
msgstr ""

#: yuminstall.py:1904
msgid "Checking dependencies in packages selected for installation"
msgstr ""

#: installclasses/fedora.py:39
msgid "_Fedora"
msgstr ""

#: installclasses/fedora.py:40
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. What additional tasks would you like your system to "
"include support for?"
msgstr ""

#: installclasses/fedora.py:49
msgid "Office and Productivity"
msgstr ""

#: installclasses/fedora.py:50 installclasses/rhel.py:58
msgid "Software Development"
msgstr ""

#: installclasses/fedora.py:51 installclasses/rhel.py:68
msgid "Web Server"
msgstr ""

#: installclasses/rhel.py:40
msgid "Red Hat Enterprise Linux"
msgstr ""

#: installclasses/rhel.py:41
#, python-format
msgid ""
"The default installation of %s is a minimal install. What additional tasks "
"would you like your system to include support for?"
msgstr ""

#: installclasses/rhel.py:49
msgid "Desktop"
msgstr ""

#: installclasses/rhel.py:75
msgid "Advanced Server"
msgstr ""

#: iw/GroupSelector.py:147
#, python-format
msgid "Packages in %s"
msgstr ""

#: iw/GroupSelector.py:424
#, python-format
msgid "Optional packages selected: %(inst)d of %(cnt)d"
msgstr ""

#: iw/GroupSelector.py:426
#, python-format
msgid "<i>%s</i>"
msgstr ""

#: iw/GroupSelector.py:486
msgid "Uncategorized"
msgstr ""

#: iw/account_gui.py:52
msgid "Root _Password:"
msgstr ""

#: iw/account_gui.py:54
msgid "_Confirm:"
msgstr ""

#: iw/account_gui.py:92
msgid "Caps Lock is on."
msgstr ""

#: iw/account_gui.py:102 iw/account_gui.py:110 iw/account_gui.py:117
#: iw/account_gui.py:137 textw/userauth_text.py:74
msgid "Error with Password"
msgstr ""

#: iw/account_gui.py:103
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: iw/account_gui.py:111
msgid "The passwords you entered were different.  Please try again."
msgstr ""

#: iw/account_gui.py:118
msgid "The root password must be at least six characters long."
msgstr ""

#: iw/account_gui.py:127 textw/userauth_text.py:83
msgid "Weak Password"
msgstr ""

#: iw/account_gui.py:128
#, python-format
msgid "You have provided a weak password: %s"
msgstr ""

#: iw/account_gui.py:130
msgid "Use Anyway"
msgstr ""

#: iw/account_gui.py:138 textw/userauth_text.py:75
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""

#: iw/autopart_type.py:100
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""

#: iw/autopart_type.py:123
msgid "Resize FileSystem Error"
msgstr ""

#: iw/autopart_type.py:124
#, python-format
msgid "%(device)s: %(msg)s"
msgstr ""

#: iw/autopart_type.py:133
msgid "Resize Device Error"
msgstr ""

#: iw/autopart_type.py:134
#, python-format
msgid "%(name)s: %(msg)s"
msgstr ""

#: iw/autopart_type.py:209
msgid ""
"Do you really want to boot from a disk which is not used for installation?"
msgstr ""

#: iw/autopart_type.py:286
msgid "Invalid Initiator Name"
msgstr ""

#: iw/autopart_type.py:287
msgid "You must provide an initiator name."
msgstr ""

#: iw/autopart_type.py:313
msgid "Error with Data"
msgstr ""

#: iw/autopart_type.py:471 iw/autopart_type.py:472
msgid "Rescanning disks"
msgstr ""

#: iw/autopart_type.py:526 textw/partition_text.py:63
msgid "Use entire drive"
msgstr ""

#: iw/autopart_type.py:527 textw/partition_text.py:64
msgid "Replace existing Linux system"
msgstr ""

#: iw/autopart_type.py:528
msgid "Shrink current system"
msgstr ""

#: iw/autopart_type.py:529 textw/partition_text.py:65
msgid "Use free space"
msgstr ""

#: iw/autopart_type.py:530
msgid "Create custom layout"
msgstr ""

#: iw/blpasswidget.py:44
msgid "_Use a boot loader password"
msgstr ""

#: iw/blpasswidget.py:45
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""

#: iw/blpasswidget.py:76
msgid "Change _password"
msgstr ""

#: iw/blpasswidget.py:99
msgid "Enter Boot Loader Password"
msgstr ""

#: iw/blpasswidget.py:105
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""

#: iw/blpasswidget.py:112
msgid "_Password:"
msgstr ""

#: iw/blpasswidget.py:118
msgid "Con_firm:"
msgstr ""

#: iw/blpasswidget.py:139
msgid "Passwords don't match"
msgstr ""

#: iw/blpasswidget.py:140
msgid "Passwords do not match"
msgstr ""

#: iw/blpasswidget.py:149
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: iw/bootloader_main_gui.py:36
msgid "Boot Loader Configuration"
msgstr ""

#: iw/bootloader_main_gui.py:179 iw/bootloader_main_gui.py:184
#: iw/bootloader_main_gui.py:221
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr ""

#: iw/bootloader_main_gui.py:227
msgid "_Change device"
msgstr ""

#: iw/congrats_gui.py:33
msgid "Congratulations"
msgstr ""

#: iw/congrats_gui.py:74 textw/complete_text.py:39
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""

#: iw/congrats_gui.py:77 textw/complete_text.py:42
msgid "Shutdown"
msgstr ""

#: iw/congrats_gui.py:79 textw/complete_text.py:44
msgid "Please shutdown to use the installed system.\n"
msgstr ""

#: iw/congrats_gui.py:81 textw/complete_text.py:46
msgid "Please reboot to use the installed system.\n"
msgstr ""

#: iw/congrats_gui.py:86 textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""

#: iw/congrats_gui.py:90 textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""

#: iw/examine_gui.py:37
msgid "Upgrade Examine"
msgstr ""

#: iw/examine_gui.py:59
#, python-format
msgid "_Install %s"
msgstr ""

#: iw/examine_gui.py:61
msgid ""
"Choose this option to freshly install your system. Existing software and "
"data may be overwritten depending on your configuration choices."
msgstr ""

#: iw/examine_gui.py:65
msgid "_Upgrade an existing installation"
msgstr ""

#: iw/examine_gui.py:67
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option preserves the existing data on your drives."
msgstr ""

#: iw/examine_gui.py:123
msgid "The following installed system will be upgraded:"
msgstr ""

#: iw/examine_gui.py:136
msgid "Unknown Linux system"
msgstr ""

#: iw/language_gui.py:33
msgid "Language Selection"
msgstr ""

#: iw/language_gui.py:81 loader/lang.c:377
msgid "What language would you like to use during the installation process?"
msgstr ""

#: iw/lvm_dialog_gui.py:124 iw/lvm_dialog_gui.py:170 iw/lvm_dialog_gui.py:184
#: iw/lvm_dialog_gui.py:223 iw/lvm_dialog_gui.py:300 iw/lvm_dialog_gui.py:679
#: iw/lvm_dialog_gui.py:700
msgid "Not enough space"
msgstr ""

#: iw/lvm_dialog_gui.py:125
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""

#: iw/lvm_dialog_gui.py:134
msgid "Confirm Physical Extent Change"
msgstr ""

#: iw/lvm_dialog_gui.py:135
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""

#: iw/lvm_dialog_gui.py:144 iw/lvm_dialog_gui.py:206
msgid "C_ontinue"
msgstr ""

#: iw/lvm_dialog_gui.py:171
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"(curpe)10.2f MB) is larger than the smallest physical volume (%(maxpvsize)"
"10.2f MB) in the volume group."
msgstr ""

#: iw/lvm_dialog_gui.py:185
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"(curpe)10.2f MB) is too large compared to the size of the smallest physical "
"volume (%(maxpvsize)10.2f MB) in the volume group."
msgstr ""

#: iw/lvm_dialog_gui.py:199
msgid "Too small"
msgstr ""

#: iw/lvm_dialog_gui.py:200
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""

#: iw/lvm_dialog_gui.py:224
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""

#: iw/lvm_dialog_gui.py:301
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""

#: iw/lvm_dialog_gui.py:400
msgid "Make Logical Volume"
msgstr ""

#: iw/lvm_dialog_gui.py:402
#, python-format
msgid "Edit Logical Volume: %s"
msgstr ""

#: iw/lvm_dialog_gui.py:433 iw/raid_dialog_gui.py:394
msgid "_File System Type:"
msgstr ""

#: iw/lvm_dialog_gui.py:439
msgid "_Logical Volume Name:"
msgstr ""

#: iw/lvm_dialog_gui.py:448 iw/partition_dialog_gui.py:431
msgid "_Size (MB):"
msgstr ""

#: iw/lvm_dialog_gui.py:455
#, python-format
msgid "(Max size is %s MB)"
msgstr ""

#: iw/lvm_dialog_gui.py:460 iw/partition_dialog_gui.py:483
#: iw/partition_ui_helpers_gui.py:329 iw/raid_dialog_gui.py:366
msgid "_Encrypt"
msgstr ""

#: iw/lvm_dialog_gui.py:469 iw/raid_dialog_gui.py:404
msgid "Original File System Type:"
msgstr ""

#: iw/lvm_dialog_gui.py:473 iw/partition_gui.py:973 iw/partition_gui.py:1114
#: iw/raid_dialog_gui.py:409
msgid "Unknown"
msgstr ""

#: iw/lvm_dialog_gui.py:477 iw/partition_dialog_gui.py:421
#: iw/raid_dialog_gui.py:415
msgid "Original File System Label:"
msgstr ""

#: iw/lvm_dialog_gui.py:482
msgid "Logical Volume Name:"
msgstr ""

#: iw/lvm_dialog_gui.py:486
msgid "Size (MB):"
msgstr ""

#: iw/lvm_dialog_gui.py:513 iw/partition_dialog_gui.py:376
#: iw/raid_dialog_gui.py:385
msgid "_Mount Point:"
msgstr ""

#: iw/lvm_dialog_gui.py:596
msgid "Illegal Logical Volume Name"
msgstr ""

#: iw/lvm_dialog_gui.py:611
msgid "Illegal logical volume name"
msgstr ""

#: iw/lvm_dialog_gui.py:612
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""

#: iw/lvm_dialog_gui.py:650 iw/partition_dialog_gui.py:123
#: iw/raid_dialog_gui.py:177
msgid "Mount point in use"
msgstr ""

#: iw/lvm_dialog_gui.py:651 iw/partition_dialog_gui.py:124
#: iw/raid_dialog_gui.py:178
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr ""

#: iw/lvm_dialog_gui.py:666
msgid "Illegal size"
msgstr ""

#: iw/lvm_dialog_gui.py:667
msgid "The requested size as entered is not a valid number greater than 0."
msgstr ""

#: iw/lvm_dialog_gui.py:680
#, python-format
msgid ""
"The current requested size (%(size)10.2f MB) is larger than the maximum "
"logical volume size (%(maxlv)10.2f MB). To increase this limit you can "
"create more Physical Volumes from unpartitioned disk space and add them to "
"this Volume Group."
msgstr ""

#: iw/lvm_dialog_gui.py:701
#, python-format
msgid ""
"The logical volumes you have configured require %(size)d MB, but the volume "
"group only has %(tempvgsize)d MB.  Please either make the volume group "
"larger or make the logical volume(s) smaller."
msgstr ""

#: iw/lvm_dialog_gui.py:794
msgid "No free slots"
msgstr ""

#: iw/lvm_dialog_gui.py:795
#, python-format
msgid "You cannot create more than %d logical volume per volume group."
msgstr ""

#: iw/lvm_dialog_gui.py:805
msgid "No free space"
msgstr ""

#: iw/lvm_dialog_gui.py:806
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""

#: iw/lvm_dialog_gui.py:841
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr ""

#: iw/lvm_dialog_gui.py:947
msgid "Invalid Volume Group Name"
msgstr ""

#: iw/lvm_dialog_gui.py:956
msgid "Name in use"
msgstr ""

#: iw/lvm_dialog_gui.py:957
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr ""

#: iw/lvm_dialog_gui.py:1207
msgid "Not enough physical volumes"
msgstr ""

#: iw/lvm_dialog_gui.py:1208
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""

#: iw/lvm_dialog_gui.py:1219
msgid "Make LVM Volume Group"
msgstr ""

#: iw/lvm_dialog_gui.py:1222
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr ""

#: iw/lvm_dialog_gui.py:1224
msgid "Edit LVM Volume Group"
msgstr ""

#: iw/lvm_dialog_gui.py:1240
msgid "_Volume Group Name:"
msgstr ""

#: iw/lvm_dialog_gui.py:1248
msgid "Volume Group Name:"
msgstr ""

#: iw/lvm_dialog_gui.py:1256
msgid "_Physical Extent:"
msgstr ""

#: iw/lvm_dialog_gui.py:1271
msgid "Physical Volumes to _Use:"
msgstr ""

#: iw/lvm_dialog_gui.py:1277
msgid "Used Space:"
msgstr ""

#: iw/lvm_dialog_gui.py:1294
msgid "Free Space:"
msgstr ""

#: iw/lvm_dialog_gui.py:1312
msgid "Total Space:"
msgstr ""

#: iw/lvm_dialog_gui.py:1350
msgid "Logical Volume Name"
msgstr ""

#: iw/lvm_dialog_gui.py:1353 iw/partition_gui.py:582
#: iw/upgrade_swap_gui.py:139 textw/upgrade_text.py:118
msgid "Mount Point"
msgstr ""

#: iw/lvm_dialog_gui.py:1356 iw/partition_gui.py:581
msgid "Size (MB)"
msgstr ""

#: iw/lvm_dialog_gui.py:1370 iw/osbootwidget.py:95
msgid "_Add"
msgstr ""

#: iw/lvm_dialog_gui.py:1373 iw/osbootwidget.py:99 iw/partition_gui.py:1763
#: iw/partition_gui.py:1773
msgid "_Edit"
msgstr ""

#: iw/lvm_dialog_gui.py:1388
msgid "_Logical Volumes"
msgstr ""

#: iw/netconfig_dialog.py:192 textw/netconfig_text.py:36
#, python-format
msgid ""
"An error occurred converting the value entered for \"%(field)s\":\n"
"%(errmsg)s"
msgstr ""

#: iw/netconfig_dialog.py:195 iw/netconfig_dialog.py:204
#: textw/netconfig_text.py:35 textw/netconfig_text.py:42
msgid "Error With Data"
msgstr ""

#: iw/netconfig_dialog.py:203
#, python-format
msgid "A value is required for the field %s."
msgstr ""

#: iw/netconfig_dialog.py:213
#, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr ""

#: iw/netconfig_dialog.py:215
msgid "Error Configuring Network"
msgstr ""

#: iw/netconfig_dialog.py:241
msgid "Dynamic IP Address"
msgstr ""

#: iw/netconfig_dialog.py:242
#, python-format
msgid "Sending request for IP address information for %s"
msgstr ""

#: iw/netconfig_dialog.py:257 iw/netconfig_dialog.py:260
#: textw/netconfig_text.py:225 textw/netconfig_text.py:228
msgid "IP Address"
msgstr ""

#: iw/netconfig_dialog.py:267 textw/netconfig_text.py:235
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr ""

#: iw/netconfig_dialog.py:268 iw/netconfig_dialog.py:274
#: iw/netconfig_dialog.py:282 iw/netconfig_dialog.py:285
#: textw/netconfig_text.py:236 textw/netconfig_text.py:242
#: textw/netconfig_text.py:250
msgid "IPv4 Network Mask"
msgstr ""

#: iw/netconfig_dialog.py:295 textw/netconfig_text.py:263
msgid "Gateway"
msgstr ""

#: iw/netconfig_dialog.py:305 textw/netconfig_text.py:273
msgid "Nameserver"
msgstr ""

#: iw/netconfig_dialog.py:314
msgid "Error configuring network device:"
msgstr ""

#: iw/network_gui.py:67 iw/network_gui.py:73
msgid "Error with Hostname"
msgstr ""

#: iw/network_gui.py:68
msgid "You must enter a valid hostname for this computer."
msgstr ""

#: iw/network_gui.py:74
#, python-format
msgid ""
"The hostname \"%(hostname)s\" is not valid for the following reason:\n"
"\n"
"%(herrors)s"
msgstr ""

#: iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr ""

#: iw/osbootwidget.py:66
msgid "Default"
msgstr ""

#: iw/osbootwidget.py:66 iw/partition_gui.py:580
msgid "Label"
msgstr ""

#: iw/osbootwidget.py:66 iw/partition_gui.py:579
msgid "Device"
msgstr ""

#: iw/osbootwidget.py:129
msgid "Image"
msgstr ""

#: iw/osbootwidget.py:136
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""

#: iw/osbootwidget.py:144
msgid "_Label"
msgstr ""

#: iw/osbootwidget.py:152
msgid "_Device"
msgstr ""

#: iw/osbootwidget.py:182
msgid "Default Boot _Target"
msgstr ""

#: iw/osbootwidget.py:211
msgid "You must specify a label for the entry"
msgstr ""

#: iw/osbootwidget.py:220
msgid "Boot label contains illegal characters"
msgstr ""

#: iw/osbootwidget.py:244
msgid "Duplicate Label"
msgstr ""

#: iw/osbootwidget.py:245
msgid "This label is already in use for another boot entry."
msgstr ""

#: iw/osbootwidget.py:258
msgid "Duplicate Device"
msgstr ""

#: iw/osbootwidget.py:259
msgid "This device is already being used for another boot entry."
msgstr ""

#: iw/osbootwidget.py:321
msgid "Cannot Delete"
msgstr ""

#: iw/osbootwidget.py:322
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""

#: iw/partition_dialog_gui.py:57
msgid "Additional Size Options"
msgstr ""

#: iw/partition_dialog_gui.py:62
msgid "_Fixed size"
msgstr ""

#: iw/partition_dialog_gui.py:64
msgid "Fill all space _up to (MB):"
msgstr ""

#: iw/partition_dialog_gui.py:74
msgid "Fill to maximum _allowable size"
msgstr ""

#: iw/partition_dialog_gui.py:346
msgid "Add Partition"
msgstr ""

#: iw/partition_dialog_gui.py:348
#, python-format
msgid "Edit Partition: %s"
msgstr ""

#: iw/partition_dialog_gui.py:385
msgid "File System _Type:"
msgstr ""

#: iw/partition_dialog_gui.py:401
msgid "Allowable _Drives:"
msgstr ""

#: iw/partition_dialog_gui.py:470
msgid "Force to be a _primary partition"
msgstr ""

#: iw/partition_gui.py:337
#, python-format
msgid "Drive %(drive)s (%(size)-0.f MB) (Model: %(model)s)"
msgstr ""

#: iw/partition_gui.py:377 iw/partition_gui.py:427 iw/partition_gui.py:488
#: iw/partition_gui.py:1116
msgid "Free"
msgstr ""

#: iw/partition_gui.py:450
#, python-format
msgid "LVM Volume Group %s (%-0.f MB)"
msgstr ""

#: iw/partition_gui.py:510
#, python-format
msgid "MD RAID ARRAY %s (%-0.f MB)"
msgstr ""

#: iw/partition_gui.py:583
msgid "Type"
msgstr ""

#: iw/partition_gui.py:584 storage/__init__.py:1637
msgid "Format"
msgstr ""

#: iw/partition_gui.py:621
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""

#: iw/partition_gui.py:623
msgid ""
"Size\n"
"(MB)"
msgstr ""

#: iw/partition_gui.py:721
msgid "Partitioning"
msgstr ""

#: iw/partition_gui.py:811
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""

#: iw/partition_gui.py:813
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr ""

#: iw/partition_gui.py:819
msgid "Partitioning Errors"
msgstr ""

#: iw/partition_gui.py:826
msgid "The partitioning scheme you requested generated the following warnings."
msgstr ""

#: iw/partition_gui.py:828
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: iw/partition_gui.py:833
msgid "Partitioning Warnings"
msgstr ""

#: iw/partition_gui.py:854
msgid "Format Warnings"
msgstr ""

#: iw/partition_gui.py:859
msgid "_Format"
msgstr ""

#: iw/partition_gui.py:883
msgid "LVM Volume Groups"
msgstr ""

#: iw/partition_gui.py:930
msgid "RAID Devices"
msgstr ""

#: iw/partition_gui.py:995 loader/hdinstall.c:222
msgid "Hard Drives"
msgstr ""

#: iw/partition_gui.py:1088
msgid "Extended"
msgstr ""

#: iw/partition_gui.py:1090
msgid "software RAID"
msgstr ""

#: iw/partition_gui.py:1309
msgid "Cannot perform any creation action"
msgstr ""

#: iw/partition_gui.py:1310
msgid ""
"Note that the creation action requires one of the following:\n"
"\n"
"* Free space in one of the Hard Drives.\n"
"* At least two free Software RAID partitions.\n"
"* At least one free physical volume (LVM) partition.\n"
"* At least one Volume Group with free space."
msgstr ""

#: iw/partition_gui.py:1418
#, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
msgstr ""

#: iw/partition_gui.py:1424
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""

#: iw/partition_gui.py:1428
#, python-format
msgid "You currently have %d software RAID partition free to use."
msgstr ""

#: iw/partition_gui.py:1432
msgid "About RAID"
msgstr ""

#: iw/partition_gui.py:1438
#, python-format
msgid ""
"Logical Volume Manager (LVM) is a 3 level construct. The fist level is made "
"up of disks or partitions formated with LVM metadata called Physical Volumes "
"(PV).  A Volume Group (VG) sits on top of one or more PVs. The VG, in turn, "
"is the base to creat one ore more Logical Volumes (LV).  Note that a VG can "
"be an aggregate of PVs from multiple physical disk.  For more information on "
"using LVM please consult the %s documentation\n"
msgstr ""

#: iw/partition_gui.py:1446
msgid ""
"To create a PV you need a partition with free space.  To create a VG you "
"need a PV that is not part of any existing VG.  To create a LV you need a VG "
"with free space.\n"
"\n"
msgstr ""

#: iw/partition_gui.py:1450
#, python-format
msgid "You currently have %d available PV free to use.\n"
msgstr ""

#: iw/partition_gui.py:1454
msgid "You currently have free space to create PVs."
msgstr ""

#: iw/partition_gui.py:1457
msgid "About LVM"
msgstr ""

#: iw/partition_gui.py:1489
msgid "Couldn't Create Drive Clone Editor"
msgstr ""

#: iw/partition_gui.py:1490
msgid "The drive clone editor could not be created for some reason."
msgstr ""

#: iw/partition_gui.py:1549 storage/partitioning.py:190
#: storage/partitioning.py:231
msgid "Error Partitioning"
msgstr ""

#: iw/partition_gui.py:1550
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr ""

#: iw/partition_gui.py:1559
#, python-format
msgid "Warning: %s."
msgstr ""

#: iw/partition_gui.py:1590
msgid "Unable To Edit"
msgstr ""

#: iw/partition_gui.py:1591
#, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""

#: iw/partition_gui.py:1762
msgid "_Create"
msgstr ""

#: iw/partition_gui.py:1765
msgid "Re_set"
msgstr ""

#: iw/partition_gui.py:1788
msgid "Please Select A Device"
msgstr ""

#: iw/partition_ui_helpers_gui.py:108 iw/partition_ui_helpers_gui.py:130
#: iw/partition_ui_helpers_gui.py:132
msgid "<Not Applicable>"
msgstr ""

#: iw/partition_ui_helpers_gui.py:310
msgid "_Format as:"
msgstr ""

#: iw/partition_ui_helpers_gui.py:332
msgid "Mi_grate filesystem to:"
msgstr ""

#: iw/partition_ui_helpers_gui.py:362
msgid "_Resize"
msgstr ""

#: iw/partition_ui_helpers_gui.py:431
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""

#: iw/progress_gui.py:37
msgid "Installing Packages"
msgstr ""

#: iw/raid_dialog_gui.py:333
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""

#: iw/raid_dialog_gui.py:347
msgid "Make RAID Device"
msgstr ""

#: iw/raid_dialog_gui.py:350
#, python-format
msgid "Edit RAID Device: %s"
msgstr ""

#: iw/raid_dialog_gui.py:352
msgid "Edit RAID Device"
msgstr ""

#: iw/raid_dialog_gui.py:423
msgid "RAID _Device:"
msgstr ""

#: iw/raid_dialog_gui.py:441
msgid "RAID _Level:"
msgstr ""

#: iw/raid_dialog_gui.py:483
msgid "_RAID Members:"
msgstr ""

#: iw/raid_dialog_gui.py:500
msgid "Number of _spares:"
msgstr ""

#: iw/raid_dialog_gui.py:510
msgid "_Format partition?"
msgstr ""

#: iw/raid_dialog_gui.py:584
msgid ""
"The source drive has no partitions to be cloned.  You must first define "
"partitions of type 'software RAID' on this drive before it can be cloned."
msgstr ""

#: iw/raid_dialog_gui.py:588 iw/raid_dialog_gui.py:594
#: iw/raid_dialog_gui.py:608 iw/raid_dialog_gui.py:621
msgid "Source Drive Error"
msgstr ""

#: iw/raid_dialog_gui.py:595
msgid ""
"The source drive you selected has partitions which are not of type 'software "
"RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""

#: iw/raid_dialog_gui.py:609
#, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to "
"the drive %s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this "
"drive can be cloned. "
msgstr ""

#: iw/raid_dialog_gui.py:622
msgid ""
"The source drive you selected has software RAID partition(s) which are "
"members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""

#: iw/raid_dialog_gui.py:636 iw/raid_dialog_gui.py:642
#: iw/raid_dialog_gui.py:658
msgid "Target Drive Error"
msgstr ""

#: iw/raid_dialog_gui.py:637
msgid "Please select the target drives for the clone operation."
msgstr ""

#: iw/raid_dialog_gui.py:643
#, python-format
msgid "The source drive %s cannot be selected as a target drive as well."
msgstr ""

#: iw/raid_dialog_gui.py:659
#, python-format
msgid ""
"The target drive %(path)s has a partition which cannot be removed for the "
"following reason:\n"
"\n"
"\"%(rc)s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""

#: iw/raid_dialog_gui.py:722
msgid "Please select a source drive."
msgstr ""

#: iw/raid_dialog_gui.py:742
#, python-format
msgid ""
"The drive %s will now be cloned to the following drives:\n"
"\n"
msgstr ""

#: iw/raid_dialog_gui.py:747
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""

#: iw/raid_dialog_gui.py:750
msgid "Final Warning"
msgstr ""

#: iw/raid_dialog_gui.py:752
msgid "Clone Drives"
msgstr ""

#: iw/raid_dialog_gui.py:761
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr ""

#: iw/raid_dialog_gui.py:784
msgid "Clone Drive Tool"
msgstr ""

#: iw/raid_dialog_gui.py:793
msgid ""
"This tool clones the layout from a partitioned source onto other similar "
"sized drives. The source must have partitions which are restricted to that "
"drive and must ONLY contain unused software RAID partitions.  EVERYTHING on "
"the target drive(s) will be destroyed.\n"
msgstr ""

#: iw/raid_dialog_gui.py:804
msgid "Source Drive:"
msgstr ""

#: iw/raid_dialog_gui.py:812
msgid "Target Drive(s):"
msgstr ""

#: iw/raid_dialog_gui.py:820
msgid "Drives"
msgstr ""

#: iw/task_gui.py:64
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""

#: iw/task_gui.py:83
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Please provide the correct information for installing %(productName)s"
msgstr ""

#: iw/task_gui.py:157
msgid "Edit Repository"
msgstr ""

#: iw/task_gui.py:181
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""

#: iw/task_gui.py:257
msgid "Invalid Proxy URL"
msgstr ""

#: iw/task_gui.py:258
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr ""

#: iw/task_gui.py:269 iw/task_gui.py:411
msgid "Invalid Repository URL"
msgstr ""

#: iw/task_gui.py:270 iw/task_gui.py:412
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr ""

#: iw/task_gui.py:291 iw/task_gui.py:421
msgid "No Media Found"
msgstr ""

#: iw/task_gui.py:292 iw/task_gui.py:422
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr ""

#: iw/task_gui.py:310 iw/task_gui.py:438
msgid "Please enter an NFS server and path."
msgstr ""

#: iw/task_gui.py:323
#, python-format
msgid ""
"The following error occurred while setting up the repository:\n"
"\n"
"%s"
msgstr ""

#: iw/task_gui.py:345
msgid "Invalid Repository Name"
msgstr ""

#: iw/task_gui.py:346
msgid "You must provide a repository name."
msgstr ""

#: iw/task_gui.py:470 ui/addrepo.glade.h:3
msgid "Add Repository"
msgstr ""

#: iw/task_gui.py:475
msgid "No Software Repos Enabled"
msgstr ""

#: iw/task_gui.py:476
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr ""

#: iw/timezone_gui.py:63 textw/timezone_text.py:95
msgid "Time Zone Selection"
msgstr ""

#: iw/upgrade_bootloader_gui.py:37 textw/upgrade_bootloader_text.py:137
msgid "Upgrade Boot Loader Configuration"
msgstr ""

#: iw/upgrade_bootloader_gui.py:123
msgid "_Update boot loader configuration"
msgstr ""

#: iw/upgrade_bootloader_gui.py:124
msgid "This will update your current boot loader."
msgstr ""

#: iw/upgrade_bootloader_gui.py:128 textw/upgrade_bootloader_text.py:108
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr ""

#: iw/upgrade_bootloader_gui.py:131 textw/upgrade_bootloader_text.py:112
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr ""

#: iw/upgrade_bootloader_gui.py:138 textw/upgrade_bootloader_text.py:121
#, python-format
msgid ""
"The installer has detected the %(type)s boot loader currently installed on %"
"(bootDev)s."
msgstr ""

#: iw/upgrade_bootloader_gui.py:142
msgid "This is the recommended option."
msgstr ""

#: iw/upgrade_bootloader_gui.py:147
msgid "_Create new boot loader configuration"
msgstr ""

#: iw/upgrade_bootloader_gui.py:149
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""

#: iw/upgrade_bootloader_gui.py:156
msgid "_Skip boot loader updating"
msgstr ""

#: iw/upgrade_bootloader_gui.py:157
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""

#: iw/upgrade_bootloader_gui.py:168
msgid "What would you like to do?"
msgstr ""

#: iw/upgrade_migratefs_gui.py:38 textw/upgrade_text.py:36
msgid "Migrate File Systems"
msgstr ""

#: iw/upgrade_migratefs_gui.py:69
#, python-format
msgid ""
"This release of %(productName)s supports an updated file system, which has "
"several benefits over the file system traditionally shipped in %(productName)"
"s.  This installation program can migrate formatted partitions without data "
"loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr ""

#: iw/upgrade_swap_gui.py:93 textw/upgrade_text.py:101
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have %"
"dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""

#: iw/upgrade_swap_gui.py:100
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""

#: iw/upgrade_swap_gui.py:112
msgid "I _want to create a swap file"
msgstr ""

#: iw/upgrade_swap_gui.py:121
msgid "Select the _partition to put the swap file on:"
msgstr ""

#: iw/upgrade_swap_gui.py:139 textw/upgrade_text.py:119
msgid "Partition"
msgstr ""

#: iw/upgrade_swap_gui.py:139
msgid "Free Space (MB)"
msgstr ""

#: iw/upgrade_swap_gui.py:157
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""

#: iw/upgrade_swap_gui.py:172
msgid "Swap file _size (MB):"
msgstr ""

#: iw/upgrade_swap_gui.py:182
msgid "I _don't want to create a swap file"
msgstr ""

#: iw/upgrade_swap_gui.py:192
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""

#: iw/upgrade_swap_gui.py:200 textw/upgrade_text.py:188
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr ""

#: iw/upgrade_swap_gui.py:207 textw/upgrade_text.py:183
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr ""

#: iw/welcome_gui.py:56 textw/welcome_text.py:36
msgid "Network Install Required"
msgstr ""

#: iw/welcome_gui.py:57 textw/welcome_text.py:37
msgid ""
"Your installation source is set to a network location, but no netork devices "
"were found on your system.  To avoid a network installation, boot with the "
"full DVD, full CD set, or do not pass a repo= parameter that specifies a "
"network source."
msgstr ""

#: iw/welcome_gui.py:67
msgid "E_xit Installer"
msgstr ""

#: iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr ""

#: iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr ""

#: iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""

#: iw/zipl_gui.py:90 textw/zipl_text.py:71
msgid "Kernel Parameters"
msgstr ""

#: iw/zipl_gui.py:93 iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr ""

#: loader/cdinstall.c:185 loader/cdinstall.c:206 loader/mediacheck.c:60
msgid "Media Check"
msgstr ""

#: loader/cdinstall.c:185 loader/cdinstall.c:188 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/method.c:326
msgid "Test"
msgstr ""

#: loader/cdinstall.c:185 loader/cdinstall.c:189
msgid "Eject Disc"
msgstr ""

#: loader/cdinstall.c:186
#, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""

#: loader/cdinstall.c:207
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""

#: loader/cdinstall.c:229
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""

#: loader/cdinstall.c:248
msgid "Disc Found"
msgstr ""

#: loader/cdinstall.c:249
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""

#: loader/cdinstall.c:339
msgid "Scanning"
msgstr ""

#: loader/cdinstall.c:339
#, c-format
msgid "Looking for installation images on CD device %s\n"
msgstr ""

#: loader/cdinstall.c:341
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr ""

#: loader/cdinstall.c:442
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""

#: loader/cdinstall.c:451
msgid "Disc Not Found"
msgstr ""

#: loader/cdinstall.c:452 loader/driverdisk.c:244 loader/driverdisk.c:275
#: loader/driverdisk.c:359 loader/driverselect.c:81 loader/driverselect.c:179
#: loader/driverselect.c:205 loader/hdinstall.c:222 loader/hdinstall.c:279
#: loader/kbd.c:119 loader/loader.c:486 loader/loader.c:506
#: loader/loader.c:1382 loader/loader.c:1415 loader/net.c:582 loader/net.c:953
#: loader/net.c:1880 loader/net.c:1902 loader/nfsinstall.c:91
#: loader/urls.c:248 storage/__init__.py:140 storage/devicetree.py:84
#: textw/constants_text.py:52
msgid "Back"
msgstr ""

#: loader/cdinstall.c:518
msgid "Cannot find kickstart file on CDROM."
msgstr ""

#: loader/copy.c:51 loader/method.c:280
#, c-format
msgid "Failed to read directory %s: %m"
msgstr ""

#: loader/driverdisk.c:139
msgid "Loading"
msgstr ""

#: loader/driverdisk.c:139
msgid "Reading driver disk"
msgstr ""

#: loader/driverdisk.c:238 loader/driverdisk.c:270
msgid "Driver Disk Source"
msgstr ""

#: loader/driverdisk.c:239
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""

#: loader/driverdisk.c:271
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""

#: loader/driverdisk.c:306
msgid "Failed to mount partition."
msgstr ""

#: loader/driverdisk.c:311
msgid "Select driver disk image"
msgstr ""

#: loader/driverdisk.c:312
msgid "Select the file which is your driver disk image."
msgstr ""

#: loader/driverdisk.c:341
msgid "Failed to load driver disk from file."
msgstr ""

#: loader/driverdisk.c:353
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr ""

#: loader/driverdisk.c:359
msgid "Insert Driver Disk"
msgstr ""

#: loader/driverdisk.c:372
msgid "Failed to mount driver disk."
msgstr ""

#: loader/driverdisk.c:380
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr ""

#: loader/driverdisk.c:443
msgid "Manually choose"
msgstr ""

#: loader/driverdisk.c:444
msgid "Load another disk"
msgstr ""

#: loader/driverdisk.c:445
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""

#: loader/driverdisk.c:483
msgid "Driver disk"
msgstr ""

#: loader/driverdisk.c:484
msgid "Do you have a driver disk?"
msgstr ""

#: loader/driverdisk.c:493
msgid "More Driver Disks?"
msgstr ""

#: loader/driverdisk.c:494
msgid "Do you wish to load any more driver disks?"
msgstr ""

#: loader/driverdisk.c:539 loader/driverdisk.c:576 loader/hdinstall.c:372
#: loader/kickstart.c:132 loader/kickstart.c:142 loader/kickstart.c:185
#: loader/kickstart.c:497 loader/modules.c:339 loader/modules.c:351
#: loader/net.c:1587 loader/net.c:1608 loader/nfsinstall.c:371
#: loader/urlinstall.c:400 loader/urlinstall.c:409 loader/urlinstall.c:418
msgid "Kickstart Error"
msgstr ""

#: loader/driverdisk.c:540
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr ""

#: loader/driverdisk.c:577
#, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s:%s"
msgstr ""

#: loader/driverselect.c:67
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""

#: loader/driverselect.c:91
msgid "Enter Module Parameters"
msgstr ""

#: loader/driverselect.c:178
msgid "No drivers found"
msgstr ""

#: loader/driverselect.c:178
msgid "Load driver disk"
msgstr ""

#: loader/driverselect.c:179
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""

#: loader/driverselect.c:197
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""

#: loader/driverselect.c:206
msgid "Specify optional module arguments"
msgstr ""

#: loader/driverselect.c:234
msgid "Select Device Driver to Load"
msgstr ""

#: loader/hdinstall.c:135
msgid ""
"An error occured finding the installation image on your hard drive.  Please "
"check your images and try again."
msgstr ""

#: loader/hdinstall.c:223
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""

#: loader/hdinstall.c:236
#, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""

#: loader/hdinstall.c:263
msgid "Directory holding image:"
msgstr ""

#: loader/hdinstall.c:291
msgid "Select Partition"
msgstr ""

#: loader/hdinstall.c:341
#, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr ""

#: loader/hdinstall.c:373
#, c-format
msgid "Bad argument to HD kickstart method command %s: %s"
msgstr ""

#: loader/hdinstall.c:443 loader/hdinstall.c:499
msgid "Cannot find kickstart file on hard drive."
msgstr ""

#: loader/hdinstall.c:486
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr ""

#: loader/kbd.c:117
msgid "Keyboard Type"
msgstr ""

#: loader/kbd.c:118
msgid "What type of keyboard do you have?"
msgstr ""

#: loader/kickstart.c:133
#, c-format
msgid "Error opening kickstart file %s: %m"
msgstr ""

#: loader/kickstart.c:143
#, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr ""

#: loader/kickstart.c:186
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr ""

#: loader/kickstart.c:285
msgid "Cannot find ks.cfg on removable media."
msgstr ""

#: loader/kickstart.c:321
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""

#: loader/kickstart.c:330
msgid "Error downloading kickstart file"
msgstr ""

#: loader/kickstart.c:498
#, c-format
msgid "Bad argument to shutdown kickstart method command %s: %s"
msgstr ""

#: loader/lang.c:63 loader/loader.c:218
#, c-format
msgid "Welcome to %s for %s"
msgstr ""

#: loader/lang.c:64
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr ""

#: loader/lang.c:65 loader/loader.c:229
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr ""

#: loader/lang.c:376
msgid "Choose a Language"
msgstr ""

#: loader/loader.c:130
msgid "Local CD/DVD"
msgstr ""

#: loader/loader.c:131
msgid "Hard drive"
msgstr ""

#: loader/loader.c:132
msgid "NFS directory"
msgstr ""

#: loader/loader.c:437 loader/loader.c:481
msgid "Update Disk Source"
msgstr ""

#: loader/loader.c:438
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""

#: loader/loader.c:482
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""

#: loader/loader.c:500
#, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr ""

#: loader/loader.c:506
msgid "Updates Disk"
msgstr ""

#: loader/loader.c:524
msgid "Failed to mount updates disk"
msgstr ""

#: loader/loader.c:529
msgid "Updates"
msgstr ""

#: loader/loader.c:529
msgid "Reading anaconda updates"
msgstr ""

#: loader/loader.c:565
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""

#: loader/loader.c:574
msgid "Error downloading updates image"
msgstr ""

#: loader/loader.c:1191
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr ""

#: loader/loader.c:1248
msgid "Media Detected"
msgstr ""

#: loader/loader.c:1249
msgid "Found local installation media"
msgstr ""

#: loader/loader.c:1374
msgid "Rescue Method"
msgstr ""

#: loader/loader.c:1375
msgid "Installation Method"
msgstr ""

#: loader/loader.c:1377
msgid "What type of media contains the rescue image?"
msgstr ""

#: loader/loader.c:1379
msgid "What type of media contains the installation image?"
msgstr ""

#: loader/loader.c:1414
msgid "No driver found"
msgstr ""

#: loader/loader.c:1414
msgid "Select driver"
msgstr ""

#: loader/loader.c:1415
msgid "Use a driver disk"
msgstr ""

#: loader/loader.c:1416
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""

#: loader/loader.c:1646
msgid "The following devices have been found on your system."
msgstr ""

#: loader/loader.c:1648
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""

#: loader/loader.c:1652
msgid "Devices"
msgstr ""

#: loader/loader.c:1653
msgid "Done"
msgstr ""

#: loader/loader.c:1654
msgid "Add Device"
msgstr ""

#: loader/loader.c:1866
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr ""

#: loader/loader.c:2283
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr ""

#: loader/loader.c:2285
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr ""

#: loader/mediacheck.c:46
#, c-format
msgid "Unable to find install image %s"
msgstr ""

#: loader/mediacheck.c:52
#, c-format
msgid "Checking \"%s\"."
msgstr ""

#: loader/mediacheck.c:54
#, c-format
msgid "Checking media."
msgstr ""

#: loader/mediacheck.c:86
msgid ""
"Unable to read the disc checksum from the primary volume descriptor.  This "
"probably means the disc was created without adding the checksum."
msgstr ""

#: loader/mediacheck.c:93
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""

#: loader/mediacheck.c:101
msgid "Success"
msgstr ""

#: loader/mediacheck.c:102
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""

#: loader/method.c:323
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""

#: loader/method.c:326
msgid "Checksum Test"
msgstr ""

#: loader/modules.c:340
#, c-format
msgid "Bad argument to device kickstart method command %s: %s"
msgstr ""

#: loader/modules.c:352
msgid "A module name must be specified for the kickstart device command."
msgstr ""

#: loader/net.c:111
msgid "Invalid Prefix"
msgstr ""

#: loader/net.c:112
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""

#: loader/net.c:465 loader/net.c:473 loader/net.c:529
#, c-format
msgid "There was an error configuring your network interface."
msgstr ""

#: loader/net.c:467
#, c-format
msgid ""
"\n"
"This cannot be corrected in cmdline mode.\n"
"Halting.\n"
msgstr ""

#: loader/net.c:472 loader/net.c:528
msgid "Network Error"
msgstr ""

#: loader/net.c:560 textw/netconfig_text.py:133
msgid "Enable IPv4 support"
msgstr ""

#: loader/net.c:574
msgid "Enable IPv6 support"
msgstr ""

#: loader/net.c:614
msgid "Configure TCP/IP"
msgstr ""

#: loader/net.c:671
msgid "Missing Protocol"
msgstr ""

#: loader/net.c:672
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr ""

#: loader/net.c:679
msgid "IPv4 Needed for NFS"
msgstr ""

#: loader/net.c:680
msgid "NFS installation method requires IPv4 support."
msgstr ""

#: loader/net.c:779
msgid "IPv4 address:"
msgstr ""

#: loader/net.c:791 loader/net.c:858 ui/netconfig.glade.h:1
msgid "/"
msgstr ""

#: loader/net.c:846
msgid "IPv6 address:"
msgstr ""

#: loader/net.c:912 textw/netconfig_text.py:160
msgid "Gateway:"
msgstr ""

#: loader/net.c:920
msgid "Name Server:"
msgstr ""

#: loader/net.c:959
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""

#: loader/net.c:979
msgid "Manual TCP/IP Configuration"
msgstr ""

#: loader/net.c:1102 loader/net.c:1109
msgid "Missing Information"
msgstr ""

#: loader/net.c:1103
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""

#: loader/net.c:1110
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr ""

#: loader/net.c:1588
#, c-format
msgid "Bad argument to kickstart network command %s: %s"
msgstr ""

#: loader/net.c:1609
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr ""

#: loader/net.c:1686
msgid "Seconds:"
msgstr ""

#: loader/net.c:1875
msgid "Networking Device"
msgstr ""

#: loader/net.c:1876
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""

#: loader/net.c:1880
msgid "Identify"
msgstr ""

#: loader/net.c:1889
msgid "You can identify the physical port for"
msgstr ""

#: loader/net.c:1891
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""

#: loader/net.c:1901
msgid "Identify NIC"
msgstr ""

#: loader/net.c:1914
msgid "Invalid Duration"
msgstr ""

#: loader/net.c:1915
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr ""

#: loader/net.c:1927
#, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr ""

#: loader/net.c:2096 loader/net.c:2100
#, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr ""

#: loader/nfsinstall.c:68
msgid "NFS server name:"
msgstr ""

#: loader/nfsinstall.c:72
#, c-format
msgid "%s directory:"
msgstr ""

#: loader/nfsinstall.c:83
#, c-format
msgid "Please enter the server name and path to your %s installation image."
msgstr ""

#: loader/nfsinstall.c:90
msgid "NFS Setup"
msgstr ""

#: loader/nfsinstall.c:285
msgid "That directory could not be mounted from the server."
msgstr ""

#: loader/nfsinstall.c:296
#, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr ""

#: loader/nfsinstall.c:372
#, c-format
msgid "Bad argument to NFS kickstart method command %s: %s"
msgstr ""

#: loader/telnetd.c:89 loader/telnetd.c:128
msgid "Telnet"
msgstr ""

#: loader/telnetd.c:89
msgid "Waiting for telnet connection."
msgstr ""

#: loader/telnetd.c:128
msgid "Running anaconda via telnet."
msgstr ""

#: loader/urlinstall.c:150
#, c-format
msgid "Unable to retrieve %s."
msgstr ""

#: loader/urlinstall.c:243
msgid "Unable to retrieve the install image."
msgstr ""

#: loader/urlinstall.c:401
#, c-format
msgid "Bad argument to Url kickstart method command %s: %s"
msgstr ""

#: loader/urlinstall.c:410
msgid "Must supply a --url argument to Url kickstart method."
msgstr ""

#: loader/urlinstall.c:419
#, c-format
msgid "Unknown Url method %s"
msgstr ""

#: loader/urls.c:157 loader/urls.c:165
msgid "Retrieving"
msgstr ""

#: loader/urls.c:251
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr ""

#: loader/urls.c:272
msgid "Enable HTTP proxy"
msgstr ""

#: loader/urls.c:287
msgid "Proxy URL"
msgstr ""

#: loader/urls.c:292
msgid "Port"
msgstr ""

#: loader/urls.c:297
msgid "Username"
msgstr ""

#: loader/urls.c:302
msgid "Password"
msgstr ""

#: loader/urls.c:321
msgid "URL Setup"
msgstr ""

#: loader/urls.c:329
msgid "You must enter a URL."
msgstr ""

#: loader/urls.c:335
msgid "URL must be either an ftp or http URL"
msgstr ""

#: loader/windows.c:65
msgid "Loading SCSI driver"
msgstr ""

#: loader/windows.c:66
#, c-format
msgid "Loading %s driver"
msgstr ""

#: storage/__init__.py:91
msgid "Unknown Device"
msgstr ""

#: storage/__init__.py:92
#, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""

#: storage/__init__.py:103
msgid "Installation cannot continue."
msgstr ""

#: storage/__init__.py:104
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""

#: storage/__init__.py:132
msgid "Encrypt device?"
msgstr ""

#: storage/__init__.py:133
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""

#: storage/__init__.py:156
msgid "Writing storage configuration to disk"
msgstr ""

#: storage/__init__.py:157
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost."
msgstr ""

#: storage/__init__.py:162
msgid "Go _back"
msgstr ""

#: storage/__init__.py:163
msgid "_Write changes to disk"
msgstr ""

#: storage/__init__.py:176
msgid "Running..."
msgstr ""

#: storage/__init__.py:177
msgid "Storing encryption keys"
msgstr ""

#: storage/__init__.py:195
#, python-format
msgid "Error storing an encryption key: %s\n"
msgstr ""

#: storage/__init__.py:312
msgid "Finding Devices"
msgstr ""

#: storage/__init__.py:313
msgid "Finding storage devices"
msgstr ""

#: storage/__init__.py:554
msgid "This partition is holding the data for the hard drive install."
msgstr ""

#: storage/__init__.py:559
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr ""

#: storage/__init__.py:565
#, python-format
msgid "This device is part of the RAID device %s."
msgstr ""

#: storage/__init__.py:568
msgid "This device is part of a RAID device."
msgstr ""

#: storage/__init__.py:573
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr ""

#: storage/__init__.py:576
msgid "This device is part of a LVM volume group."
msgstr ""

#: storage/__init__.py:592
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""

#: storage/__init__.py:861
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: storage/__init__.py:866
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""

#: storage/__init__.py:872
#, python-format
msgid ""
"Your / partition is less than %(min)s MB which is lower than recommended for "
"a normal %(productName)s install."
msgstr ""

#: storage/__init__.py:882
#, python-format
msgid ""
"Your / partition does not match the the live image you are installing from.  "
"It must be formatted as %s."
msgstr ""

#: storage/__init__.py:889
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s megabytes which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: storage/__init__.py:918
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""

#: storage/__init__.py:921
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""

#: storage/__init__.py:928
msgid ""
"You have not specified a swap partition.  Due to the amount of memory "
"present, a swap partition is required to complete installation."
msgstr ""

#: storage/__init__.py:933
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: storage/__init__.py:940
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: storage/__init__.py:944
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr ""

#: storage/__init__.py:955
msgid "No Drives Found"
msgstr ""

#: storage/__init__.py:956
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""

#: storage/__init__.py:1188 storage/__init__.py:1197
msgid "Dirty File Systems"
msgstr ""

#: storage/__init__.py:1189
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""

#: storage/__init__.py:1198
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: storage/__init__.py:1670
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for "
"swap space, you must reformat as a new-style Linux swap partition."
msgstr ""

#: storage/__init__.py:1681
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""

#: storage/__init__.py:1689
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""

#: storage/__init__.py:1701
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a supported swap volume.  In order to continue "
"installation, you will need to format the device or skip it."
msgstr ""

#: storage/__init__.py:1712
#, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""

#: storage/__init__.py:1719
#, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""

#: storage/__init__.py:1780 storage/__init__.py:1792
msgid "Invalid mount point"
msgstr ""

#: storage/__init__.py:1781
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: storage/__init__.py:1793
#, python-format
msgid ""
"An error occurred when trying to create %(mountpoint)s: %(msg)s.  This is a "
"fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: storage/__init__.py:1808 storage/__init__.py:1831
msgid "Unable to mount filesystem"
msgstr ""

#: storage/__init__.py:1809
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s.  You may "
"continue installation, but there may be problems."
msgstr ""

#: storage/__init__.py:1832
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s: %(msg)s. This "
"is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: storage/devicelibs/lvm.py:314
#, python-format
msgid "vginfo failed for %s"
msgstr ""

#: storage/devicelibs/lvm.py:343
#, python-format
msgid "lvs failed for %s"
msgstr ""

#: storage/devicetree.py:93
msgid "Confirm"
msgstr ""

#: storage/devicetree.py:94
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""

#: storage/devicetree.py:145
#, python-format
msgid ""
"Error processing drive:\n"
"\n"
"%(path)s\n"
"%(size)-0.fMB\n"
"%(description)s\n"
"\n"
"This device may need to be reinitialized.\n"
"\n"
"REINITIALIZING WILL CAUSE ALL DATA TO BE LOST!%(details)s"
msgstr ""

#: storage/devicetree.py:152
msgid "_Ignore drive"
msgstr ""

#: storage/devicetree.py:153
msgid "_Re-initialize drive"
msgstr ""

#: storage/devicetree.py:173
#, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %(msg)s.  You can reinitialize all related "
"PVs (%(pvs)s) which will erase the LVM metadata, or ignore which will "
"preserve the contents."
msgstr ""

#: storage/devicetree.py:179
msgid "_Ignore"
msgstr ""

#: storage/devicetree.py:180
msgid "_Re-initialize"
msgstr ""

#: storage/devicetree.py:1815 storage/devicetree.py:1864
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr ""

#: storage/formats/fs.py:97
msgid "filesystem configuration missing a type"
msgstr ""

#: storage/formats/fs.py:327
msgid "Formatting"
msgstr ""

#: storage/formats/fs.py:328
#, python-format
msgid "Creating filesystem on %s"
msgstr ""

#: storage/formats/fs.py:427
msgid "Resizing"
msgstr ""

#: storage/formats/fs.py:428
#, python-format
msgid "Resizing filesystem on %s"
msgstr ""

#: storage/formats/fs.py:471
msgid "Checking"
msgstr ""

#: storage/formats/fs.py:472
#, python-format
msgid "Checking filesystem on %s"
msgstr ""

#: storage/iscsi.py:82 storage/iscsi.py:83
msgid "Scanning iSCSI nodes"
msgstr ""

#: storage/iscsi.py:160 storage/iscsi.py:161
msgid "Initializing iSCSI initiator"
msgstr ""

#: storage/iscsi.py:197
msgid "iSCSI not available"
msgstr ""

#: storage/iscsi.py:199
msgid "No initiator name set"
msgstr ""

#: storage/iscsi.py:213
msgid "No iSCSI nodes discovered"
msgstr ""

#: storage/iscsi.py:216 storage/iscsi.py:217
msgid "Logging in to iSCSI nodes"
msgstr ""

#: storage/iscsi.py:239
msgid "No new iSCSI nodes discovered"
msgstr ""

#: storage/iscsi.py:242
msgid "Could not log in to any of the discovered nodes"
msgstr ""

#: storage/partitioning.py:184
msgid ""
"Could not find enough free space for automatic partitioning.  Press 'OK' to "
"exit the installer."
msgstr ""

#: storage/partitioning.py:187
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""

#: storage/partitioning.py:216
msgid "Warnings During Automatic Partitioning"
msgstr ""

#: storage/partitioning.py:218
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""

#: storage/partitioning.py:230 storage/partitioning.py:251
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""

#: storage/partitioning.py:232
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%(msg)s.%(extra)s"
msgstr ""

#: storage/partitioning.py:253
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""

#: storage/partitioning.py:255
msgid "Automatic Partitioning Errors"
msgstr ""

#: storage/partitioning.py:256
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %(extra)s"
msgstr ""

#: storage/partitioning.py:267
msgid "Unrecoverable Error"
msgstr ""

#: storage/zfcp.py:50
msgid "You have not specified a device number or the number is invalid"
msgstr ""

#: storage/zfcp.py:52
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr ""

#: storage/zfcp.py:54
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr ""

#: storage/zfcp.py:131
#, python-format
msgid "Could not free zFCP device %(devnum)s from device ignore list (%(e)s)."
msgstr ""

#: storage/zfcp.py:137
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr ""

#: storage/zfcp.py:149
#, python-format
msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
msgstr ""

#: storage/zfcp.py:160
#, python-format
msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:167
#, python-format
msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
msgstr ""

#: storage/zfcp.py:182
#, python-format
msgid ""
"Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)"
"s)."
msgstr ""

#: storage/zfcp.py:188
#, python-format
msgid ""
"LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already configured."
msgstr ""

#: storage/zfcp.py:200
#, python-format
msgid ""
"Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on zFCP "
"device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:209
#, python-format
msgid ""
"Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
"again."
msgstr ""

#: storage/zfcp.py:266
#, python-format
msgid ""
"Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s %(fcplun)"
"s (%(e)s)."
msgstr ""

#: storage/zfcp.py:275
#, python-format
msgid ""
"Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s (%"
"(e)s)."
msgstr ""

#: storage/zfcp.py:293
#, python-format
msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:319
#, python-format
msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
msgstr ""

#: textw/complete_text.py:32
msgid "Reboot"
msgstr ""

#: textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr ""

#: textw/complete_text.py:62
msgid "Complete"
msgstr ""

#: textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr ""

#: textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr ""

#: textw/netconfig_text.py:43
#, python-format
msgid "A value is required for the field %s"
msgstr ""

#: textw/netconfig_text.py:88 ui/netconfig.glade.h:9
msgid "Enable network interface"
msgstr ""

#: textw/netconfig_text.py:91 ui/netconfig.glade.h:10
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""

#: textw/netconfig_text.py:130
msgid "Use dynamic IP configuration (DHCP)"
msgstr ""

#: textw/netconfig_text.py:140
msgid "IPv4 Address:"
msgstr ""

#: textw/netconfig_text.py:163
msgid "Nameserver:"
msgstr ""

#: textw/netconfig_text.py:193
msgid "Missing Device"
msgstr ""

#: textw/netconfig_text.py:194
msgid "You must select a network device"
msgstr ""

#: textw/netconfig_text.py:253
msgid "IPv4 Network Mask "
msgstr ""

#: textw/netconfig_text.py:276
msgid "Configuring Network Interfaces"
msgstr ""

#: textw/netconfig_text.py:276
msgid "Waiting for NetworkManager"
msgstr ""

#: textw/netconfig_text.py:282
msgid "Error configuring network device"
msgstr ""

#: textw/netconfig_text.py:282
#, python-format
msgid "Error configuring network device %s"
msgstr ""

#: textw/partition_text.py:58
msgid "Partitioning Type"
msgstr ""

#: textw/partition_text.py:60
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target."
msgstr ""

#: textw/partition_text.py:80
msgid "Which drive(s) do you want to use for this installation?"
msgstr ""

#: textw/partition_text.py:95
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr ""

#: textw/partition_text.py:173 ui/adddrive.glade.h:4
msgid "Advanced Storage Options"
msgstr ""

#: textw/partition_text.py:174 ui/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr ""

#: textw/partition_text.py:203
msgid "Add FCP Device"
msgstr ""

#: textw/partition_text.py:204 ui/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""

#: textw/partition_text.py:223
msgid "Add FCoE SAN"
msgstr ""

#: textw/partition_text.py:224
msgid ""
"Enter the device name for the NIC which is connected to the FCoE SAN. For "
"example \"eth0\"."
msgstr ""

#: textw/partition_text.py:225
msgid "NIC device name"
msgstr ""

#: textw/partition_text.py:232
#, python-format
msgid "%s is not a valid NIC device name."
msgstr ""

#: textw/partition_text.py:248 ui/iscsi-config.glade.h:7
msgid "Configure iSCSI Parameters"
msgstr ""

#: textw/partition_text.py:249 ui/iscsi-config.glade.h:8
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: textw/partition_text.py:250
msgid "Target IP Address"
msgstr ""

#: textw/partition_text.py:251
msgid "iSCSI Initiator Name"
msgstr ""

#: textw/partition_text.py:252
msgid "CHAP username"
msgstr ""

#: textw/partition_text.py:253
msgid "CHAP password"
msgstr ""

#: textw/partition_text.py:254
msgid "Reverse CHAP username"
msgstr ""

#: textw/partition_text.py:255
msgid "Reverse CHAP password"
msgstr ""

#: textw/progress_text.py:46
msgid "Package Installation"
msgstr ""

#: textw/timezone_text.py:74
msgid "In which time zone are you located?"
msgstr ""

#: textw/timezone_text.py:92
msgid "System clock uses UTC"
msgstr ""

#: textw/upgrade_bootloader_text.py:116 textw/upgrade_bootloader_text.py:126
msgid "Update boot loader configuration"
msgstr ""

#: textw/upgrade_bootloader_text.py:129
msgid "Skip boot loader updating"
msgstr ""

#: textw/upgrade_bootloader_text.py:131
msgid "Create new boot loader configuration"
msgstr ""

#: textw/upgrade_text.py:38
#, python-format
msgid ""
"This release of %(productName)s supports the an updated file system, which "
"has several benefits over the file system traditionally shipped in %"
"(productName)s.  This installation program can migrate formatted partitions "
"without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: textw/upgrade_text.py:119
msgid "Free Space"
msgstr ""

#: textw/upgrade_text.py:137
msgid "RAM detected (MB):"
msgstr ""

#: textw/upgrade_text.py:140
msgid "Suggested size (MB):"
msgstr ""

#: textw/upgrade_text.py:143
msgid "Swap file size (MB):"
msgstr ""

#: textw/upgrade_text.py:151
msgid "Add Swap"
msgstr ""

#: textw/upgrade_text.py:176
msgid "The value you entered is not a valid number."
msgstr ""

#: textw/upgrade_text.py:209
msgid "Reinstall System"
msgstr ""

#: textw/upgrade_text.py:219
msgid "System to Upgrade"
msgstr ""

#: textw/upgrade_text.py:220
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""

#: textw/userauth_text.py:30
msgid "Root Password"
msgstr ""

#: textw/userauth_text.py:33
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""

#: textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr ""

#: textw/userauth_text.py:84
#, python-format
msgid ""
"You have provided a weak password: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr ""

#: textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""

#: textw/zipl_text.py:35
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""

#: textw/zipl_text.py:67
msgid "z/IPL Configuration"
msgstr ""

#: textw/zipl_text.py:75 textw/zipl_text.py:79
msgid "Chandev line "
msgstr ""

#: liveinst/liveinst.desktop.in.h:1
msgid "Install"
msgstr ""

#: liveinst/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr ""

#: liveinst/liveinst.desktop.in.h:3
msgid "Install to Hard Drive"
msgstr ""

#: ui/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""

#: ui/GroupSelector.glade.h:2
msgid "_Deselect"
msgstr ""

#: ui/GroupSelector.glade.h:3
msgid "_Deselect all optional packages"
msgstr ""

#: ui/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr ""

#: ui/GroupSelector.glade.h:5
msgid "_Select"
msgstr ""

#: ui/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr ""

#: ui/GroupSelector.glade.h:7
msgid "dialog1"
msgstr ""

#: ui/account.glade.h:1
msgid "Confirm:"
msgstr ""

#: ui/account.glade.h:2
msgid "Root Password:"
msgstr ""

#: ui/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: ui/adddrive.glade.h:1
msgid "Add _FCoE SAN"
msgstr ""

#: ui/adddrive.glade.h:2
msgid "Add _ZFCP LUN"
msgstr ""

#: ui/adddrive.glade.h:3
msgid "Add _iSCSI target"
msgstr ""

#: ui/adddrive.glade.h:6
msgid "_Add drive"
msgstr ""

#: ui/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr ""

#: ui/addrepo.glade.h:2
msgid "<b>Repository _type:</b>"
msgstr ""

#: ui/addrepo.glade.h:4
msgid "Configure _proxy"
msgstr ""

#: ui/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""

#: ui/addrepo.glade.h:9
msgid ""
"Please provide the configuration information for this software repository."
msgstr ""

#: ui/addrepo.glade.h:10
msgid "Proxy U_RL (host:port)"
msgstr ""

#: ui/addrepo.glade.h:11
msgid "Proxy pass_word"
msgstr ""

#: ui/addrepo.glade.h:12
msgid "Proxy u_sername"
msgstr ""

#: ui/addrepo.glade.h:13
msgid "Repository _URL"
msgstr ""

#: ui/addrepo.glade.h:14
msgid "Select A Directory"
msgstr ""

#: ui/addrepo.glade.h:15
msgid "URL is a _mirror list"
msgstr ""

#: ui/addrepo.glade.h:16
msgid "_Directory"
msgstr ""

#: ui/addrepo.glade.h:17
msgid "_Options"
msgstr ""

#: ui/addrepo.glade.h:18
msgid "_Partition"
msgstr ""

#: ui/addrepo.glade.h:19
msgid "_Path"
msgstr ""

#: ui/addrepo.glade.h:20
msgid "_Server"
msgstr ""

#: ui/anaconda.glade.h:1
msgid "Reboo_t"
msgstr ""

#: ui/anaconda.glade.h:3
msgid "_Next"
msgstr ""

#: ui/autopart.glade.h:1
msgid "<b>Shrink _target (in MB):</b>"
msgstr ""

#: ui/autopart.glade.h:2
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target. You can also choose to create your own custom "
"layout."
msgstr ""

#: ui/autopart.glade.h:3
msgid "Re_view and modify partitioning layout"
msgstr ""

#: ui/autopart.glade.h:4
msgid "Volume to Shrink"
msgstr ""

#: ui/autopart.glade.h:5
msgid "What drive would you like to _boot this installation from?"
msgstr ""

#: ui/autopart.glade.h:6
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr ""

#: ui/autopart.glade.h:7
msgid "_Advanced storage configuration"
msgstr ""

#: ui/autopart.glade.h:8
msgid "_Encrypt system"
msgstr ""

#: ui/autopart.glade.h:9
msgid "_Select the drive(s) to use for this installation."
msgstr ""

#: ui/autopart.glade.h:10
msgid "_Shrink"
msgstr ""

#: ui/blwhere.glade.h:1
msgid "/boot"
msgstr ""

#: ui/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr ""

#: ui/blwhere.glade.h:3
msgid "Boot loader device"
msgstr ""

#: ui/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:6
msgid "MBR"
msgstr ""

#: ui/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:9
msgid "Where would you like to install the boot loader for your system?"
msgstr ""

#: ui/detailed-dialog.glade.h:1
msgid "Info"
msgstr ""

#: ui/detailed-dialog.glade.h:2
msgid "_Details"
msgstr ""

#: ui/fcoe-config.glade.h:1
msgid "Configure FCoE Parameters"
msgstr ""

#: ui/fcoe-config.glade.h:2
msgid "NIC"
msgstr ""

#: ui/fcoe-config.glade.h:3
msgid ""
"Please select the network interface which is connected to\n"
"your FCoE switch."
msgstr ""

#: ui/fcoe-config.glade.h:5
msgid "_Add FCoE Disk(s)"
msgstr ""

#: ui/iscsi-config.glade.h:1
msgid "<b>CHAP _Password:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:2
msgid "<b>CHAP _Username:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:3
msgid "<b>Reverse CHAP P_assword:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:4
msgid "<b>Reverse CHAP U_sername:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:5
msgid "<b>_Target IP Address:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:6
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:9
msgid "_Add target"
msgstr ""

#: ui/lukspassphrase.glade.h:1
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""

#: ui/lukspassphrase.glade.h:2
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""

#: ui/lukspassphrase.glade.h:3
msgid "Confirm passphrase:"
msgstr ""

#: ui/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr ""

#: ui/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr ""

#: ui/netconfig.glade.h:2
msgid "<b>Gateway:</b>"
msgstr ""

#: ui/netconfig.glade.h:3
msgid "<b>IPv4 Address:</b>"
msgstr ""

#: ui/netconfig.glade.h:4
msgid "<b>IPv6 Address:</b>"
msgstr ""

#: ui/netconfig.glade.h:5
msgid "<b>Nameserver:</b>"
msgstr ""

#: ui/netconfig.glade.h:6
msgid "<b>_Interface:</b>"
msgstr ""

#: ui/netconfig.glade.h:7
msgid "Enable IPv_4 support"
msgstr ""

#: ui/netconfig.glade.h:8
msgid "Enable IPv_6 support"
msgstr ""

#: ui/netconfig.glade.h:11
msgid "Use _dynamic IP configuration (DHCP)"
msgstr ""

#: ui/network.glade.h:1
msgid "Hostname:"
msgstr ""

#: ui/network.glade.h:2
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""

#: ui/tasksel.glade.h:1
msgid "Customize _later"
msgstr ""

#: ui/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""

#: ui/tasksel.glade.h:4
#, no-c-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""

#: ui/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""

#: ui/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr ""

#: ui/tasksel.glade.h:7
msgid "_Customize now"
msgstr ""

#: ui/tasksel.glade.h:8
msgid "_Modify repository"
msgstr ""

#: ui/zfcp-config.glade.h:1
msgid "<b>Device number:</b>"
msgstr ""

#: ui/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr ""

#: ui/zfcp-config.glade.h:3
msgid "<b>WWPN:</b>"
msgstr ""

#: ui/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr ""

#: lang-table.h:1
msgid "Afrikaans"
msgstr ""

#: lang-table.h:2
msgid "Arabic"
msgstr ""

#: lang-table.h:3
msgid "Assamese"
msgstr ""

#: lang-table.h:4
msgid "Bengali"
msgstr ""

#: lang-table.h:5
msgid "Bengali(India)"
msgstr ""

#: lang-table.h:6
msgid "Bulgarian"
msgstr ""

#: lang-table.h:7
msgid "Catalan"
msgstr ""

#: lang-table.h:8
msgid "Chinese(Simplified)"
msgstr ""

#: lang-table.h:9
msgid "Chinese(Traditional)"
msgstr ""

#: lang-table.h:10
msgid "Croatian"
msgstr ""

#: lang-table.h:11
msgid "Czech"
msgstr ""

#: lang-table.h:12
msgid "Danish"
msgstr ""

#: lang-table.h:13
msgid "Dutch"
msgstr ""

#: lang-table.h:14
msgid "English"
msgstr ""

#: lang-table.h:15
msgid "Estonian"
msgstr ""

#: lang-table.h:16
msgid "Finnish"
msgstr ""

#: lang-table.h:17
msgid "French"
msgstr ""

#: lang-table.h:18
msgid "German"
msgstr ""

#: lang-table.h:19
msgid "Greek"
msgstr ""

#: lang-table.h:20
msgid "Gujarati"
msgstr ""

#: lang-table.h:21
msgid "Hebrew"
msgstr ""

#: lang-table.h:22
msgid "Hindi"
msgstr ""

#: lang-table.h:23
msgid "Hungarian"
msgstr ""

#: lang-table.h:24
msgid "Icelandic"
msgstr ""

#: lang-table.h:25
msgid "Iloko"
msgstr ""

#: lang-table.h:26
msgid "Indonesian"
msgstr ""

#: lang-table.h:27
msgid "Italian"
msgstr ""

#: lang-table.h:28
msgid "Japanese"
msgstr ""

#: lang-table.h:29
msgid "Kannada"
msgstr ""

#: lang-table.h:30
msgid "Korean"
msgstr ""

#: lang-table.h:31
msgid "Macedonian"
msgstr ""

#: lang-table.h:32
msgid "Maithili"
msgstr ""

#: lang-table.h:33
msgid "Malay"
msgstr ""

#: lang-table.h:34
msgid "Malayalam"
msgstr ""

#: lang-table.h:35
msgid "Marathi"
msgstr ""

#: lang-table.h:36
msgid "Nepali"
msgstr ""

#: lang-table.h:37
msgid "Norwegian(Bokmål)"
msgstr ""

#: lang-table.h:38
msgid "Northern Sotho"
msgstr ""

#: lang-table.h:39
msgid "Oriya"
msgstr ""

#: lang-table.h:40
msgid "Persian"
msgstr ""

#: lang-table.h:41
msgid "Polish"
msgstr ""

#: lang-table.h:42
msgid "Portuguese"
msgstr ""

#: lang-table.h:43
msgid "Portuguese(Brazilian)"
msgstr ""

#: lang-table.h:44
msgid "Punjabi"
msgstr ""

#: lang-table.h:45
msgid "Romanian"
msgstr ""

#: lang-table.h:46
msgid "Russian"
msgstr ""

#: lang-table.h:47
msgid "Serbian"
msgstr ""

#: lang-table.h:48
msgid "Serbian(Latin)"
msgstr ""

#: lang-table.h:49
msgid "Sinhala"
msgstr ""

#: lang-table.h:50
msgid "Slovak"
msgstr ""

#: lang-table.h:51
msgid "Slovenian"
msgstr ""

#: lang-table.h:52
msgid "Spanish"
msgstr ""

#: lang-table.h:53
msgid "Swedish"
msgstr ""

#: lang-table.h:54
msgid "Tajik"
msgstr ""

#: lang-table.h:55
msgid "Tamil"
msgstr ""

#: lang-table.h:56
msgid "Telugu"
msgstr ""

#: lang-table.h:57
msgid "Turkish"
msgstr ""

#: lang-table.h:58
msgid "Ukrainian"
msgstr ""

#: lang-table.h:59
msgid "Vietnamese"
msgstr ""

#: lang-table.h:60
msgid "Welsh"
msgstr ""

#: lang-table.h:61
msgid "Zulu"
msgstr ""
